<!DOCTYPE html>
<!-- saved from url=(0057)https://en.wikipedia.org/wiki/Multiple_sequence_alignment -->
<html class="client-js ve-not-available" lang="en" dir="ltr"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

<title>Multiple sequence alignment - Wikipedia</title>
<script>document.documentElement.className = document.documentElement.className.replace( /(^|\s)client-nojs(\s|$)/, "$1client-js$2" );</script>
<script>(window.RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":false,"wgNamespaceNumber":0,"wgPageName":"Multiple_sequence_alignment","wgTitle":"Multiple sequence alignment","wgCurRevisionId":774102939,"wgRevisionId":774102939,"wgArticleId":4066308,"wgIsArticle":true,"wgIsRedirect":false,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":["CS1 maint: Multiple names: authors list","Good articles","Bioinformatics","Computational phylogenetics","Markov models"],"wgBreakFrames":false,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"Multiple_sequence_alignment","wgRelevantArticleId":4066308,"wgRequestId":"WYYlDwpAAD4AADx-jtgAAADI","wgIsProbablyEditable":true,"wgRelevantPageIsProbablyEditable":true,"wgRestrictionEdit":[],"wgRestrictionMove":[],"wgFlaggedRevsParams":{"tags":{}},"wgStableRevisionId":null,"wgWikiEditorEnabledModules":{"toolbar":true,"preview":false,"publish":false},"wgBetaFeaturesFeatures":[],"wgMediaViewerOnClick":true,"wgMediaViewerEnabledByDefault":false,"wgPopupsShouldSendModuleToUser":false,"wgPopupsConflictsWithNavPopupGadget":false,"wgVisualEditor":{"pageLanguageCode":"en","pageLanguageDir":"ltr","pageVariantFallbacks":"en","usePageImages":true,"usePageDescriptions":true},"wgPreferredVariant":"en","wgMFExpandAllSectionsUserOption":false,"wgMFDisplayWikibaseDescriptions":{"search":true,"nearby":true,"watchlist":true,"tagline":false},"wgRelatedArticles":null,"wgRelatedArticlesUseCirrusSearch":true,"wgRelatedArticlesOnlyUseCirrusSearch":false,"wgULSCurrentAutonym":"English","wgNoticeProject":"wikipedia","wgCentralNoticeCookiesToDelete":[],"wgCentralNoticeCategoriesUsingLegacy":["Fundraising","fundraising"],"wgCategoryTreePageCategoryOptions":"{\"mode\":0,\"hideprefix\":20,\"showcount\":true,\"namespaces\":false}","wgWikibaseItemId":"Q1377767","wgCentralAuthMobileDomain":false,"wgVisualEditorToolbarScrollOffset":0,"wgVisualEditorUnsupportedEditParams":["preload","preloadparams","preloadtitle","undo","undoafter","veswitched"],"wgEditSubmitButtonLabelPublish":false});mw.loader.state({"ext.gadget.charinsert-styles":"ready","ext.globalCssJs.user.styles":"ready","ext.globalCssJs.site.styles":"ready","site.styles":"ready","noscript":"ready","user.styles":"ready","user":"ready","user.options":"loading","user.tokens":"loading","ext.cite.styles":"ready","wikibase.client.init":"ready","ext.visualEditor.desktopArticleTarget.noscript":"ready","ext.uls.interlanguage":"ready","mediawiki.legacy.shared":"ready","mediawiki.legacy.commonPrint":"ready","mediawiki.sectionAnchor":"ready","mediawiki.skinning.interface":"ready","skins.vector.styles":"ready","ext.globalCssJs.user":"ready","ext.globalCssJs.site":"ready"});mw.loader.implement("user.options@0bhc5ha",function($,jQuery,require,module){mw.user.options.set([]);});mw.loader.implement("user.tokens@1dqfd7l",function ( $, jQuery, require, module ) {
mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\","csrfToken":"+\\"});/*@nomin*/

});mw.loader.load(["ext.cite.a11y","site","mediawiki.page.startup","mediawiki.user","mediawiki.hidpi","mediawiki.page.ready","mediawiki.toc","mediawiki.searchSuggest","ext.gadget.teahouse","ext.gadget.ReferenceTooltips","ext.gadget.watchlist-notice","ext.gadget.DRN-wizard","ext.gadget.charinsert","ext.gadget.refToolbar","ext.gadget.extra-toolbar-buttons","ext.gadget.switcher","ext.gadget.featured-articles-links","ext.centralauth.centralautologin","mmv.head","mmv.bootstrap.autostart","ext.visualEditor.desktopArticleTarget.init","ext.visualEditor.targetLoader","ext.eventLogging.subscriber","ext.wikimediaEvents","ext.navigationTiming","ext.uls.eventlogger","ext.uls.init","ext.uls.interface","ext.centralNotice.geoIP","ext.centralNotice.startUp","skins.vector.js"]);});</script>
<link rel="stylesheet" href="./Multiple sequence alignment - Wikipedia_files/load.php">
<script async="" src="./Multiple sequence alignment - Wikipedia_files/load(1).php"></script>
<style>
.cite-accessibility-label{ top:-99999px;clip:rect( 1px 1px 1px 1px ); clip:rect( 1px,1px,1px,1px ); position:absolute !important;padding:0 !important;border:0 !important;height:1px !important;width:1px !important; overflow:hidden}
@media screen {
	.tochidden,.toctoggle{-moz-user-select:none;-webkit-user-select:none;-ms-user-select:none;user-select:none}.toctoggle{font-size:94%}}
@media print {
	.toc.tochidden,.toctoggle{display:none}}
.suggestions{overflow:hidden;position:absolute;top:0;left:0;width:0;border:0;z-index:1099;padding:0;margin:-1px 0 0 0}.suggestions-special{position:relative;background-color:#fff;cursor:pointer;border:solid 1px #aaa;margin:0;margin-top:-2px;display:none;padding:0.25em 0.25em;line-height:1.25em}.suggestions-results{background-color:#fff;cursor:pointer;border:solid 1px #aaa;padding:0;margin:0}.suggestions-result{color:#000;margin:0;line-height:1.5em;padding:0.01em 0.25em;text-align:left; overflow:hidden;-o-text-overflow:ellipsis; text-overflow:ellipsis;white-space:nowrap}.suggestions-result-current{background-color:#4c59a6;color:#fff}.suggestions-special .special-label{color:#808080;text-align:left}.suggestions-special .special-query{color:#000;font-style:italic;text-align:left}.suggestions-special .special-hover{background-color:#c0c0c0}.suggestions-result-current .special-label,.suggestions-result-current .special-query{color:#fff}.highlight{font-weight:bold}
.wp-teahouse-question-form{position:absolute;margin-left:auto;margin-right:auto;background-color:#f4f3f0;border:1px solid #a7d7f9;padding:1em}#wp-th-question-ask{float:right}.wp-teahouse-ask a.external{background-image:none !important}.wp-teahouse-respond-form{position:absolute;margin-left:auto;margin-right:auto;background-color:#f4f3f0;border:1px solid #a7d7f9;padding:1em}.wp-th-respond{float:right}.wp-teahouse-respond a.external{background-image:none !important}
.referencetooltip{position:absolute;list-style:none;list-style-image:none;opacity:0;font-size:10px;margin:0;z-index:5;padding:0}.referencetooltip li{border:#080086 2px solid;max-width:260px;padding:10px 8px 13px 8px;margin:0px;background-color:#F7F7F7;-webkit-box-shadow:2px 4px 2px rgba(0,0,0,0.3);-moz-box-shadow:2px 4px 2px rgba(0,0,0,0.3);box-shadow:2px 4px 2px rgba(0,0,0,0.3)}.referencetooltip li+li{margin-left:7px;margin-top:-2px;border:0;padding:0;height:3px;width:0px;background-color:transparent;-webkit-box-shadow:none;-moz-box-shadow:none;box-shadow:none;border-top:12px #080086 solid;border-right:7px transparent solid;border-left:7px transparent solid}.referencetooltip>li+li::after{content:'';border-top:8px #F7F7F7 solid;border-right:5px transparent solid;border-left:5px transparent solid;margin-top:-12px;margin-left:-5px;z-index:1;height:0px;width:0px;display:block}.client-js body .referencetooltip li li{border:none;-webkit-box-shadow:none;-moz-box-shadow:none;box-shadow:none;height:auto;width:auto;margin:auto;padding:0;position:static}.RTflipped{padding-top:13px}.referencetooltip.RTflipped li+li{position:absolute;top:2px;border-top:0;border-bottom:12px #080086 solid}.referencetooltip.RTflipped li+li::after{border-top:0;border-bottom:8px #F7F7F7 solid;position:absolute;margin-top:7px}.RTsettings{float:right;height:24px;width:24px;cursor:pointer;background-image:url(//upload.wikimedia.org/wikipedia/commons/thumb/0/05/OOjs_UI_icon_advanced.svg/24px-OOjs_UI_icon_advanced.svg.png);background-image:linear-gradient(transparent,transparent),url(//upload.wikimedia.org/wikipedia/commons/0/05/OOjs_UI_icon_advanced.svg);margin-top:-9px;margin-right:-7px;-webkit-transition:opacity 0.15s;-moz-transition:opacity 0.15s;-ms-transition:opacity 0.15s;-o-transition:opacity 0.15s;transition:opacity 0.15s;opacity:0.6;filter:alpha(opacity=60)}.RTsettings:hover{opacity:1;filter:alpha(opacity=100)}.RTTarget{border:#080086 2px solid}
.skin-vector li.GA,.skin-monobook li.GA,.skin-modern li.GA{list-style-image:url(//upload.wikimedia.org/wikipedia/commons/4/42/Monobook-bullet-ga.png)} .skin-vector li.FA,.skin-monobook li.FA{list-style-image:url(//upload.wikimedia.org/wikipedia/commons/d/d4/Monobook-bullet-star.png)}.skin-modern li.FA{list-style-image:url(//upload.wikimedia.org/wikipedia/commons/thumb/2/2c/Modern-bullet-star.svg/9px-Modern-bullet-star.svg.png)}
@-webkit-keyframes centralAuthPPersonalAnimation{0%{opacity:0;-webkit-transform:translateY( -20px )}100%{opacity:1;-webkit-transform:translateY( 0 )}}@-moz-keyframes centralAuthPPersonalAnimation{0%{opacity:0;-moz-transform:translateY( -20px )}100%{opacity:1;-moz-transform:translateY( 0 )}}@-o-keyframes centralAuthPPersonalAnimation{0%{opacity:0;-o-transform:translateY( -20px )}100%{opacity:1;-o-transform:translateY( 0 )}}@keyframes centralAuthPPersonalAnimation{0%{opacity:0;transform:translateY( -20px )}100%{opacity:1;transform:translateY( 0 )}}.centralAuthPPersonalAnimation{-webkit-animation-duration:1s;-moz-animation-duration:1s;-o-animation-duration:1s;animation-duration:1s;-webkit-animation-fill-mode:both;-moz-animation-fill-mode:both;-o-animation-fill-mode:both;animation-fill-mode:both;-webkit-animation-name:centralAuthPPersonalAnimation;-moz-animation-name:centralAuthPPersonalAnimation;-o-animation-name:centralAuthPPersonalAnimation;animation-name:centralAuthPPersonalAnimation}
.mw-ui-button{font-family:inherit;font-size:1em;display:inline-block;min-width:4em;max-width:28.75em;padding:0.546875em 1em;line-height:1.286;margin:0;border-radius:2px;-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;-webkit-appearance:none;*display:inline; zoom:1;vertical-align:middle;background-color:#f8f9fa;color:#222222;border:1px solid #a2a9b1;text-align:center;font-weight:bold;cursor:pointer}.mw-ui-button:visited{color:#222222}.mw-ui-button:hover{background-color:#ffffff;color:#444444;border-color:#a2a9b1}.mw-ui-button:focus{background-color:#ffffff;color:#222222;border-color:#3366cc;box-shadow:inset 0 0 0 1px #3366cc,inset 0 0 0 2px #ffffff}.mw-ui-button:active,.mw-ui-button.is-on,.mw-ui-button.mw-ui-checked{background-color:#d9d9d9;color:#000000;border-color:#72777d;box-shadow:none}.mw-ui-button:disabled{background-color:#c8ccd1;color:#fff;border-color:#c8ccd1}.mw-ui-button:disabled:hover,.mw-ui-button:disabled:active{background-color:#c8ccd1;color:#fff;box-shadow:none;border-color:#c8ccd1}.mw-ui-button:focus{outline-width:0}.mw-ui-button:focus::-moz-focus-inner{border-color:transparent;padding:0}.mw-ui-button:not( :disabled ){-webkit-transition:background-color 100ms,color 100ms,border-color 100ms,box-shadow 100ms;-moz-transition:background-color 100ms,color 100ms,border-color 100ms,box-shadow 100ms;transition:background-color 100ms,color 100ms,border-color 100ms,box-shadow 100ms}.mw-ui-button:disabled{text-shadow:none;cursor:default}.mw-ui-button.mw-ui-big{font-size:1.3em}.mw-ui-button.mw-ui-block{display:block;width:100%;margin-left:auto;margin-right:auto}.mw-ui-button.mw-ui-progressive{background-color:#3366cc;color:#fff;border:1px solid #3366cc;text-shadow:0 1px rgba(0,0,0,0.1)}.mw-ui-button.mw-ui-progressive:hover{background-color:#447ff5;border-color:#447ff5}.mw-ui-button.mw-ui-progressive:focus{box-shadow:inset 0 0 0 1px #3366cc,inset 0 0 0 2px #ffffff}.mw-ui-button.mw-ui-progressive:active,.mw-ui-button.mw-ui-progressive.is-on,.mw-ui-button.mw-ui-progressive.mw-ui-checked{background-color:#2a4b8d;border-color:#2a4b8d;box-shadow:none}.mw-ui-button.mw-ui-progressive:disabled{background-color:#c8ccd1;color:#fff;border-color:#c8ccd1}.mw-ui-button.mw-ui-progressive:disabled:hover,.mw-ui-button.mw-ui-progressive:disabled:active,.mw-ui-button.mw-ui-progressive:disabled.mw-ui-checked{background-color:#c8ccd1;color:#fff;border-color:#c8ccd1;box-shadow:none}.mw-ui-button.mw-ui-progressive.mw-ui-quiet{color:#222222}.mw-ui-button.mw-ui-progressive.mw-ui-quiet:hover{background-color:transparent;color:#447ff5}.mw-ui-button.mw-ui-progressive.mw-ui-quiet:active,.mw-ui-button.mw-ui-progressive.mw-ui-quiet.mw-ui-checked{color:#2a4b8d}.mw-ui-button.mw-ui-progressive.mw-ui-quiet:focus{background-color:transparent;color:#3366cc}.mw-ui-button.mw-ui-progressive.mw-ui-quiet:disabled{color:#c8ccd1}.mw-ui-button.mw-ui-destructive{background-color:#dd3333;color:#fff;border:1px solid #dd3333;text-shadow:0 1px rgba(0,0,0,0.1)}.mw-ui-button.mw-ui-destructive:hover{background-color:#ff4242;border-color:#ff4242}.mw-ui-button.mw-ui-destructive:focus{box-shadow:inset 0 0 0 1px #dd3333,inset 0 0 0 2px #ffffff}.mw-ui-button.mw-ui-destructive:active,.mw-ui-button.mw-ui-destructive.is-on,.mw-ui-button.mw-ui-destructive.mw-ui-checked{background-color:#b32424;border-color:#b32424;box-shadow:none}.mw-ui-button.mw-ui-destructive:disabled{background-color:#c8ccd1;color:#fff;border-color:#c8ccd1}.mw-ui-button.mw-ui-destructive:disabled:hover,.mw-ui-button.mw-ui-destructive:disabled:active,.mw-ui-button.mw-ui-destructive:disabled.mw-ui-checked{background-color:#c8ccd1;color:#fff;border-color:#c8ccd1;box-shadow:none}.mw-ui-button.mw-ui-destructive.mw-ui-quiet{color:#222222}.mw-ui-button.mw-ui-destructive.mw-ui-quiet:hover{background-color:transparent;color:#ff4242}.mw-ui-button.mw-ui-destructive.mw-ui-quiet:active,.mw-ui-button.mw-ui-destructive.mw-ui-quiet.mw-ui-checked{color:#b32424}.mw-ui-button.mw-ui-destructive.mw-ui-quiet:focus{background-color:transparent;color:#dd3333}.mw-ui-button.mw-ui-destructive.mw-ui-quiet:disabled{color:#c8ccd1}.mw-ui-button.mw-ui-quiet{background:transparent;border:0;text-shadow:none;color:#222222}.mw-ui-button.mw-ui-quiet:hover{background-color:transparent;color:#444444}.mw-ui-button.mw-ui-quiet:active,.mw-ui-button.mw-ui-quiet.mw-ui-checked{color:#000000}.mw-ui-button.mw-ui-quiet:focus{background-color:transparent;color:#222222}.mw-ui-button.mw-ui-quiet:disabled{color:#c8ccd1}.mw-ui-button.mw-ui-quiet:hover,.mw-ui-button.mw-ui-quiet:focus{box-shadow:none}.mw-ui-button.mw-ui-quiet:active,.mw-ui-button.mw-ui-quiet:disabled{background:transparent}input.mw-ui-button::-moz-focus-inner,button.mw-ui-button::-moz-focus-inner{margin-top:-1px;margin-bottom:-1px}a.mw-ui-button{text-decoration:none}a.mw-ui-button:hover,a.mw-ui-button:focus{text-decoration:none}.mw-ui-button-group > *{min-width:48px;border-radius:0;float:left}.mw-ui-button-group > *:first-child{border-top-left-radius:2px;border-bottom-left-radius:2px}.mw-ui-button-group > *:not( :first-child ){border-left:0}.mw-ui-button-group > *:last-child{border-top-right-radius:2px;border-bottom-right-radius:2px}.mw-ui-button-group .is-on .button{cursor:default}
.mw-ui-icon{position:relative;line-height:1.5em;min-height:1.5em;min-width:1.5em}.mw-ui-icon.mw-ui-icon-element{text-indent:-999px;overflow:hidden;width:3.5em;min-width:3.5em;max-width:3.5em}.mw-ui-icon.mw-ui-icon-element:before{left:0;right:0;position:absolute;margin:0 1em}.mw-ui-icon.mw-ui-icon-before:before,.mw-ui-icon.mw-ui-icon-element:before{background-position:50% 50%;background-repeat:no-repeat;background-size:100% auto;float:left;display:block;min-height:1.5em;content:''}.mw-ui-icon.mw-ui-icon-before:before{position:relative;width:1.5em;margin-right:1em}.mw-ui-icon.mw-ui-icon-small:before{background-size:66.67% auto}
.mw-notification-area{position:absolute}.mw-notification-area-floating{position:fixed}.mw-notification-area{top:0;right:0; padding:1em;width:20em;line-height:1.35;z-index:10000; overflow:hidden}.mw-notification{padding:0.25em 1em;margin-bottom:0.5em;border:solid 1px #ddd;background-color:#fcfcfc; cursor:pointer;opacity:0;-webkit-transform:translateX( 35px );transform:translateX( 35px );-webkit-transition:opacity 0.35s ease-in-out,-webkit-transform 0.35s ease-in-out;transition:opacity 0.35s ease-in-out,transform 0.35s ease-in-out}.mw-notification-visible{opacity:1;-webkit-transform:translateX( 0 );transform:translateX( 0 )}.mw-notification-replaced{opacity:0;-webkit-transform:translateY( -35px );transform:translateY( -35px );pointer-events:none}.mw-notification-title{font-weight:bold}.mw-notification-type-warn{border-color:#f5be00; background-color:#ffffe8}.mw-notification-type-error{border-color:#eb3941; background-color:#fff8f8} .mw-notification-area{font-size:0.8em}.mw-notification-area-layout{top:7em}.mw-notification{background-color:#fff;padding:0.75em 1.5em;border:1px solid #a2a9b1;border-radius:2px;box-shadow:0 2px 2px 0 rgba(0,0,0,0.25)}.mw-notification-type-warn{border-color:#f5be00; }.mw-notification-type-error{border-color:#d33; }
@media print {
	.mw-notification-area{display:none}}
.uls-menu{border-radius:4px; font-size:medium}.uls-search,.uls-language-settings-close-block{border-top-right-radius:4px;border-top-left-radius:4px}.uls-language-list{border-bottom-right-radius:4px;border-bottom-left-radius:4px} .uls-menu.callout .caret-before,.uls-menu.callout .caret-after{border-top:10px solid transparent;border-bottom:10px solid transparent;display:inline-block; top:17px;position:absolute}.uls-menu.callout .caret-after{display:inline-block}.uls-menu.callout .caret-right .caret-before,.uls-menu.callout .caret-right .caret-after{ border-left:10px solid #c9c9c9; right:-11px}.uls-menu.callout .caret-right .caret-after{ border-left:10px solid #fcfcfc; right:-10px}.uls-menu.callout--languageselection .caret-right .caret-after{ border-left:10px solid #fff}.uls-menu.callout .caret-left .caret-before,.uls-menu.callout .caret-left .caret-after{ border-right:10px solid #c9c9c9; left:-11px}.uls-menu.callout .caret-left .caret-after{ border-right:10px solid #fcfcfc; left:-10px}.uls-menu.callout--languageselection .caret-left .caret-after{ border-right:10px solid #fff}.uls-ui-languages button{margin:5px 15px 5px 0;white-space:nowrap;overflow:hidden}.uls-search-wrapper-wrapper{position:relative;padding-left:40px;margin-top:5px;margin-bottom:5px}.uls-icon-back{background:transparent url(/w/extensions/UniversalLanguageSelector/resources/images/back-grey-ltr.png?90e9b) no-repeat scroll center center;background-image:-webkit-linear-gradient(transparent,transparent),url(/w/extensions/UniversalLanguageSelector/resources/images/back-grey-ltr.svg?ae714);background-image:linear-gradient(transparent,transparent),url(data:image/svg+xml,%3C%3Fxml%20version%3D%221.0%22%20encoding%3D%22UTF-8%22%20standalone%3D%22no%22%3F%3E%0A%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2024%2024%22%20id%3D%22Layer_1%22%3E%0A%20%20%20%20%3Cpath%20d%3D%22M7%2013.1l8.9%208.9c.8-.8.8-2%200-2.8l-6.1-6.1%206-6.1c.8-.8.8-2%200-2.8L7%2013.1z%22%20id%3D%22path3%22%20fill%3D%22%23555%22%2F%3E%0A%3C%2Fsvg%3E%0A);background-size:28px;background-position:center center;height:32px;width:40px;display:block;position:absolute;left:0;border-right:1px solid #c9c9c9;opacity:0.8}.uls-icon-back:hover{opacity:1;cursor:pointer}
@media print{#centralNotice{display:none}}.cn-closeButton{display:inline-block;zoom:1;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABMAAAATCAYAAAByUDbMAAACeklEQVR4Aa1UM3jkcRBN6qS//tTFttPHqmI359VZ5dm2L1zFtr02YlRx5sX2ft/7ed7O/w1M5ufnt8NZQjCBQXhG+IYZe5zjfju7xcHc3NzEzMwM6xOEqNnZ2ZeVlZW827dv1ycmJnbGx8d3Yr5582Z9eXk5D/d4h/empqYm+K0nw3yKcF4sFmdzOJzGgIAAhb29vc7Ozk6/Atrr/fz8lCwWq6m3tzcb72G3nmzFo/NNTU354eHhIltbW72Tk5M2IyND9P79+8ZPnz7VvXnzpoHBYPS4uLhobGxsDMHBwaLa2tp82MF+PVmUUqnMioiI6LOysjLQLCcPS2ZmZn7Ozc39oPtFYG80GgWpqakSS0tLY1hYmEgikWTBfoXsLD16BT3gUWhoqEKtVheREZ+Qu0IEjI+PZ2k0GqFer+cnJSWJra2tDWw2u2FqauoVeEAW3NzcnBcYGCh3dHTUtba2ltNjLvRJSEiQEQEPRENDQzkMBqPXwsKiXyqVFsFDNzc3LWmoqKmp4YIHZIyHDx9WOzg46GJiYmTk5e/h4eHstLQ0CX2ykXST4JPJ8y7sSVM5/QGMf9y4caMHf3r//v1a8IDsGRm04/D58+dtpNFPPNRqtflXrlzpI8G15IEGc2xsrFSlUglwD+Tk5NRBmuTk5A7wgOxbXFxcF8iysrIa1mskEolKvL29Ne7u7mpXV1dNaWlp9fr7X79+VYMMeQieY/dsv5p1r2g2Nja2vWZ7RXNiYuIA0dwlzwYGBjbkGXmURXeLeUa1ul2ebV8BEH+7CiAiASTYvgJ2qc3MzMzF2vz8+XPd27dvG5hMZg+CsV1tHmfXOP5+dqyddgHOI7v1srTdcwAAAABJRU5ErkJggg==);background:url(/w/extensions/CentralNotice/resources/subscribing/CloseWindow19x19.png?7596b)!ie;width:19px;height:19px;text-indent:19px;white-space:nowrap;overflow:hidden}</style><style>
.suggestions a.mw-searchSuggest-link,.suggestions a.mw-searchSuggest-link:hover,.suggestions a.mw-searchSuggest-link:active,.suggestions a.mw-searchSuggest-link:focus{color:#000;text-decoration:none}.suggestions-result-current a.mw-searchSuggest-link,.suggestions-result-current a.mw-searchSuggest-link:hover,.suggestions-result-current a.mw-searchSuggest-link:active,.suggestions-result-current a.mw-searchSuggest-link:focus{color:#fff}.suggestions a.mw-searchSuggest-link .special-query{ overflow:hidden;-o-text-overflow:ellipsis; text-overflow:ellipsis;white-space:nowrap}
.mw-mmv-overlay{position:fixed;top:0;left:0;right:0;bottom:0;z-index:1000;background-color:#000}body.mw-mmv-lightbox-open{overflow-y:auto;  }body.mw-mmv-lightbox-open #mw-page-base,body.mw-mmv-lightbox-open #mw-head-base,body.mw-mmv-lightbox-open #mw-navigation,body.mw-mmv-lightbox-open #content,body.mw-mmv-lightbox-open #footer,body.mw-mmv-lightbox-open #globalWrapper{ display:none}body.mw-mmv-lightbox-open > *{ display:none}body.mw-mmv-lightbox-open > .mw-mmv-overlay,body.mw-mmv-lightbox-open > .mw-mmv-wrapper{display:block}.mw-mmv-filepage-buttons{margin-top:5px}.mw-mmv-filepage-buttons .mw-mmv-view-expanded,.mw-mmv-filepage-buttons .mw-mmv-view-config{display:block;line-height:inherit}.mw-mmv-filepage-buttons .mw-mmv-view-expanded.mw-ui-icon:before{background-image:url(data:image/svg+xml,%3C%3Fxml%20version%3D%221.0%22%20encoding%3D%22UTF-8%22%20standalone%3D%22no%22%3F%3E%0A%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%201024%20768%22%3E%0A%20%20%20%20%3Cg%20fill%3D%22%2372777d%22%3E%0A%20%20%20%20%20%20%20%20%3Cpath%20d%3D%22M851.2%2071.6L690.7%20232.1l-40.1-40.3-9.6%20164.8%20164.8-9.3-40.3-40.4L926%20146.4l58.5%2058.5L997.6%200%20792.7%2013.1%22%2F%3E%0A%20%20%20%20%20%20%20%20%3Cpath%20d%3D%22M769.6%2089.3H611.9l70.9%2070.8%207.9%207.5m-47.1%20234.6l-51.2%203%203-51.2%209.4-164.4%205.8-100.3H26.4V768h883.1V387l-100.9%205.8-165%209.4zM813.9%20678H113.6l207.2-270.2%2031.5-12.9L548%20599.8l105.9-63.2%20159.8%20140.8.2.6zm95.6-291.9V228l-79.1%2078.9%207.8%207.9%22%2F%3E%0A%20%20%20%20%3C%2Fg%3E%0A%3C%2Fsvg%3E%0A)}.mw-mmv-filepage-buttons .mw-mmv-view-config.mw-ui-icon:before{background-image:url(data:image/svg+xml,%3C%3Fxml%20version%3D%221.0%22%20encoding%3D%22UTF-8%22%20standalone%3D%22no%22%3F%3E%0A%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%201024%20768%22%3E%0A%20%20%20%20%3Cpath%20d%3D%22M897%20454.6V313.4L810.4%20299c-6.4-23.3-16-45.7-27.3-65.8l50.5-71.4-99.4-100.2-71.4%2050.5c-20.9-11.2-42.5-20.9-65.8-27.3L582.6-1H441.4L427%2085.6c-23.3%206.4-45.7%2016-65.8%2027.3l-71.4-50.5-100.3%2099.5%2050.5%2071.4c-11.2%2020.9-20.9%2042.5-27.3%2066.6L127%20313.4v141.2l85.8%2014.4c6.4%2023.3%2016%2045.7%2027.3%2066.6L189.6%20607l99.5%2099.5%2071.4-50.5c20.9%2011.2%2042.5%2020.9%2066.6%2027.3l14.4%2085.8h141.2l14.4-86.6c23.3-6.4%2045.7-16%2065.8-27.3l71.4%2050.5%2099.5-99.5-50.5-71.4c11.2-20.9%2020.9-42.5%2027.3-66.6l86.4-13.6zm-385%2077c-81.8%200-147.6-66.6-147.6-147.6%200-81.8%2066.6-147.6%20147.6-147.6S659.6%20302.2%20659.6%20384%20593.8%20531.6%20512%20531.6z%22%20fill%3D%22%2372777d%22%2F%3E%0A%3C%2Fsvg%3E%0A);opacity:0.75}.mw-mmv-filepage-buttons .mw-mmv-view-config.mw-ui-icon:before:hover{opacity:1}
.postedit-container{margin:0 auto;position:fixed;top:0;height:0;left:50%;z-index:1000;font-size:13px}.postedit-container:hover{cursor:pointer}.postedit{position:relative;top:0.6em;left:-50%;line-height:1.35;opacity:1;-webkit-transition:opacity 250ms;-moz-transition:opacity 250ms;transition:opacity 250ms}.postedit.mw-notification{padding-right:3em}.skin-monobook .postedit{top:6em !important; }.postedit-faded{opacity:0}.postedit-icon{padding-left:41px;  line-height:25px;background-repeat:no-repeat;background-position:8px 50%}.postedit-icon-checkmark{background-image:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAB9ElEQVR4AZWRA3AYURQArxrVHtW2bdu2bdu2zdi2bdu2bWxs7zeehZaw4f70kbs+zI3e/nWK+RWx3aOFlrL56Sy5SxrruG69hlv6OyK+mz+8KDSXdXembj0ispT7tjs4ZTIbpYBvxGSGKzZTeFrb7W/meN002swFs0U8ttpHTkF2BvCqWQrW35929bTsKm5Zb+SEwWwcY8wAngB9m7Z+d+rIPZ/npdy12M5p47n8dXsCYAf0qPy06eGMdktuDu9Qf+JmKl3SWM91qzVcN9tAbEYkwMaq0tyb1m/To5kP170el/BK8/qa6sJr70ydf+T/Uu5ab+Oo/lS0AkUBpIFWlZ9WPhxpse/PHO7YbOOczjL0vZV2lNxPPtG73dYXM+xvm2znrOl83tidoqCwMBgYXsPFB0on5S6pr+eK5TKuW67lgvaKvF8mL1dtfTL32FHxRdyx3cQpg7m4x9sCXKkTIzA4LDH44zWdzaUf71hv5rTG4uyzcusybxSX7aThbMQ8XgCYAp3rzTTQOiIh9PNlzY3FSuZxrzjme1Y7uGS6kjsWO4jPjM4FVjRZsvD4kO9XtTZzQn82NyzWc0B7AmZh6gA/hOYSGhfw9YbOVnarj+S7800AL2BIsxUAbWNToj7bhBuQmZcOsFdoKUC74rGheCwXmqAIQTc9jQcrADIAAAAASUVORK5CYII=);background-image:url(/w/resources/src/mediawiki.action/images/green-checkmark.png?d94f1)!ie;background-position:left}.postedit-close{position:absolute;padding:0 0.8em;right:0;top:0;font-size:1.25em;font-weight:bold;line-height:2.3em;color:#000;text-shadow:0 0.0625em 0 #fff;text-decoration:none;opacity:0.2}.postedit:hover .postedit-close{color:#000;text-decoration:none;opacity:0.4}
#uls-settings-block{border-top:1px solid #c9c9c9;background:#f8f8f8;background:-webkit-gradient( linear,left top,left bottom,from( #fbfbfb ),to( #f0f0f0 ) );background:-webkit-linear-gradient( top,#fbfbfb,#f0f0f0 );background:-moz-linear-gradient( top,#fbfbfb,#f0f0f0 );background:-o-linear-gradient( top,#fbfbfb,#f0f0f0 );background:linear-gradient( #fbfbfb,#f0f0f0 );padding-left:10px;line-height:1.2em;border-radius:0 0 5px 5px}#uls-settings-block div.display-settings-block,#uls-settings-block div.input-settings-block{display:inline-block;margin:8px 15px;color:#565656}#uls-settings-block div.display-settings-block:hover,#uls-settings-block div.input-settings-block:hover{color:#252525}</style><style>
.ve-activated #toc,.ve-activated #siteNotice,.ve-activated .mw-indicators,.ve-activated #t-print,.ve-activated #t-permalink,.ve-activated #p-coll-print_export,.ve-activated #t-cite,.ve-deactivating .ve-ui-surface,.ve-active .ve-init-mw-desktopArticleTarget-editableContent{display:none} .ve-activating .ve-ui-surface{height:0;padding:0 !important; overflow:hidden} .ve-loading #content > :not( .ve-init-mw-desktopArticleTarget-loading-overlay ), .ve-activated .ve-init-mw-desktopArticleTarget-uneditableContent{ pointer-events:none;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;opacity:0.5}.ve-activated #catlinks{cursor:pointer}.ve-activated #catlinks a{opacity:1} .ve-activated #content{position:relative} .ve-init-mw-desktopArticleTarget-loading-overlay{position:absolute;left:0;right:0;z-index:1;margin-top:-0.5em}.ve-init-mw-desktopArticleTarget-progress{height:1em;overflow:hidden;margin:0 25%}.ve-init-mw-desktopArticleTarget-progress-bar{height:1em;width:0} .mw-editsection{white-space:nowrap; unicode-bidi:-moz-isolate;unicode-bidi:-webkit-isolate;unicode-bidi:isolate}.mw-editsection-divider{color:#555} .ve-init-mw-desktopArticleTarget-progress{height:0.75em;border:1px solid #36c;background:#fff;border-radius:2px;box-shadow:0 0.1em 0 0 rgba( 0,0,0,0.15 )}.ve-init-mw-desktopArticleTarget-progress-bar{height:0.75em;background:#36c}</style><meta name="ResourceLoaderDynamicStyles" content="">
<link rel="stylesheet" href="./Multiple sequence alignment - Wikipedia_files/load(2).php">
<link rel="stylesheet" href="./Multiple sequence alignment - Wikipedia_files/load(3).php">
<meta name="generator" content="MediaWiki 1.30.0-wmf.12">
<meta name="referrer" content="origin-when-cross-origin">
<meta property="og:image" content="https://upload.wikimedia.org/wikipedia/commons/thumb/7/79/RPLP0_90_ClustalW_aln.gif/1200px-RPLP0_90_ClustalW_aln.gif">
<link rel="alternate" href="android-app://org.wikipedia/http/en.m.wikipedia.org/wiki/Multiple_sequence_alignment">
<link rel="alternate" type="application/x-wiki" title="Edit this page" href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit">
<link rel="edit" title="Edit this page" href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit">
<link rel="apple-touch-icon" href="https://en.wikipedia.org/static/apple-touch/wikipedia.png">
<link rel="shortcut icon" href="https://en.wikipedia.org/static/favicon/wikipedia.ico">
<link rel="search" type="application/opensearchdescription+xml" href="https://en.wikipedia.org/w/opensearch_desc.php" title="Wikipedia (en)">
<link rel="EditURI" type="application/rsd+xml" href="https://en.wikipedia.org/w/api.php?action=rsd">
<link rel="license" href="https://creativecommons.org/licenses/by-sa/3.0/">
<link rel="canonical" href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment">
<link rel="dns-prefetch" href="https://login.wikimedia.org/">
<link rel="dns-prefetch" href="https://meta.wikimedia.org/">
<script src="./Multiple sequence alignment - Wikipedia_files/load(4).php"></script></head>
<body class="mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject page-Multiple_sequence_alignment rootpage-Multiple_sequence_alignment skin-vector action-view">		<div id="mw-page-base" class="noprint"></div>
		<div id="mw-head-base" class="noprint"></div>
		<div id="content" class="mw-body" role="main"><div id="mw-notification-area" class="mw-notification-area mw-notification-area-layout" style="display: none;"></div>
			<a id="top"></a>

							<div id="siteNotice" class="mw-body-content"><div id="centralNotice"></div><!-- CentralNotice --></div>
						<div class="mw-indicators mw-body-content">
<div id="mw-indicator-good-star" class="mw-indicator"><a href="https://en.wikipedia.org/wiki/Wikipedia:Good_articles" title="This is a good article. Click here for more information."><img alt="This is a good article. Click here for more information." src="./Multiple sequence alignment - Wikipedia_files/19px-Symbol_support_vote.svg.png" width="19" height="20" srcset="//upload.wikimedia.org/wikipedia/en/thumb/9/94/Symbol_support_vote.svg/29px-Symbol_support_vote.svg.png 1.5x, //upload.wikimedia.org/wikipedia/en/thumb/9/94/Symbol_support_vote.svg/39px-Symbol_support_vote.svg.png 2x" data-file-width="180" data-file-height="185"></a></div>
</div>
			<h1 id="firstHeading" class="firstHeading" lang="en">Multiple sequence alignment</h1>
									<div id="bodyContent" class="mw-body-content">
									<div id="siteSub" class="noprint">From Wikipedia, the free encyclopedia</div>
								<div id="contentSub"></div>
												<div id="jump-to-nav" class="mw-jump">
					Jump to:					<a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#mw-head">navigation</a>, 					<a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#p-search">search</a>
				</div>
				<div id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><div class="mw-parser-output"><div class="thumb tright">
<div class="thumbinner" style="width:577px;"><a href="https://en.wikipedia.org/wiki/File:RPLP0_90_ClustalW_aln.gif" class="image"><img alt="" src="./Multiple sequence alignment - Wikipedia_files/575px-RPLP0_90_ClustalW_aln.gif" width="575" height="325" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/7/79/RPLP0_90_ClustalW_aln.gif/863px-RPLP0_90_ClustalW_aln.gif 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/7/79/RPLP0_90_ClustalW_aln.gif/1150px-RPLP0_90_ClustalW_aln.gif 2x" data-file-width="1250" data-file-height="706"></a>
<div class="thumbcaption">
<div class="magnify"><a href="https://en.wikipedia.org/wiki/File:RPLP0_90_ClustalW_aln.gif" class="internal" title="Enlarge"></a></div>
First 90 positions of a protein multiple sequence alignment of instances of the acidic ribosomal protein P0 (L10E) from several organisms. Generated with <a href="https://en.wikipedia.org/wiki/ClustalX" class="mw-redirect" title="ClustalX">ClustalX</a>.</div>
</div>
</div>
<p>A <b>multiple sequence alignment</b> (<b>MSA</b>) is a <a href="https://en.wikipedia.org/wiki/Sequence_alignment" title="Sequence alignment">sequence alignment</a> of three or more <a href="https://en.wikipedia.org/wiki/Biological_sequence" class="mw-redirect" title="Biological sequence">biological sequences</a>, generally <a href="https://en.wikipedia.org/wiki/Protein" title="Protein">protein</a>, <a href="https://en.wikipedia.org/wiki/DNA" title="DNA">DNA</a>, or <a href="https://en.wikipedia.org/wiki/RNA" title="RNA">RNA</a>. In many cases, the input set of query sequences are assumed to have an <a href="https://en.wikipedia.org/wiki/Evolutionary" class="mw-redirect" title="Evolutionary">evolutionary</a> relationship by which they share a lineage and are descended from a common ancestor. From the resulting MSA, sequence <a href="https://en.wikipedia.org/wiki/Homology_(biology)" title="Homology (biology)">homology</a> can be inferred and <a href="https://en.wikipedia.org/wiki/Molecular_phylogeny" class="mw-redirect" title="Molecular phylogeny">phylogenetic analysis</a> can be conducted to assess the sequences' shared evolutionary origins. Visual depictions of the alignment as in the image at right illustrate <a href="https://en.wikipedia.org/wiki/Mutation" title="Mutation">mutation</a> events such as point mutations (single <a href="https://en.wikipedia.org/wiki/Amino_acid" title="Amino acid">amino acid</a> or <a href="https://en.wikipedia.org/wiki/Nucleotide" title="Nucleotide">nucleotide</a> changes) that appear as differing characters in a single alignment column, and insertion or deletion mutations (<a href="https://en.wikipedia.org/wiki/Indel" title="Indel">indels</a> or gaps) that appear as hyphens in one or more of the sequences in the alignment. Multiple sequence alignment is often used to assess sequence <a href="https://en.wikipedia.org/wiki/Conservation_(genetics)" class="mw-redirect" title="Conservation (genetics)">conservation</a> of <a href="https://en.wikipedia.org/wiki/Protein_domain" title="Protein domain">protein domains</a>, <a href="https://en.wikipedia.org/wiki/Tertiary_structure" class="mw-redirect" title="Tertiary structure">tertiary</a> and <a href="https://en.wikipedia.org/wiki/Secondary_structure" class="mw-redirect" title="Secondary structure">secondary</a> structures, and even individual amino acids or nucleotides.</p>
<p>Multiple sequence alignment also refers to the process of aligning such a sequence set. Because three or more sequences of biologically relevant length can be difficult and are almost always time-consuming to align by hand, computational <a href="https://en.wikipedia.org/wiki/Algorithm" title="Algorithm">algorithms</a> are used to produce and analyze the alignments. MSAs require more sophisticated methodologies than <a href="https://en.wikipedia.org/wiki/Sequence_alignment" title="Sequence alignment">pairwise alignment</a> because they are more <a href="https://en.wikipedia.org/wiki/Computational_complexity_theory" title="Computational complexity theory">computationally complex</a>. Most multiple sequence alignment programs use <a href="https://en.wikipedia.org/wiki/Heuristic" title="Heuristic">heuristic</a> methods rather than <a href="https://en.wikipedia.org/wiki/Global_optimization" title="Global optimization">global optimization</a> because identifying the optimal alignment between more than a few sequences of moderate length is prohibitively computationally expensive.</p>
<p></p>
<div id="toc" class="toc">
<div class="toctitle">
<h2>Contents</h2>
<span class="toctoggle">&nbsp;[<a role="button" tabindex="0" class="togglelink">hide</a>]&nbsp;</span></div>
<ul>
<li class="toclevel-1 tocsection-1"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Dynamic_programming_and_computational_complexity"><span class="tocnumber">1</span> <span class="toctext">Dynamic programming and computational complexity</span></a></li>
<li class="toclevel-1 tocsection-2"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Progressive_alignment_construction"><span class="tocnumber">2</span> <span class="toctext">Progressive alignment construction</span></a></li>
<li class="toclevel-1 tocsection-3"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Iterative_methods"><span class="tocnumber">3</span> <span class="toctext">Iterative methods</span></a></li>
<li class="toclevel-1 tocsection-4"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Consensus_methods"><span class="tocnumber">4</span> <span class="toctext">Consensus methods</span></a></li>
<li class="toclevel-1 tocsection-5"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Hidden_Markov_models"><span class="tocnumber">5</span> <span class="toctext">Hidden Markov models</span></a></li>
<li class="toclevel-1 tocsection-6"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Genetic_algorithms_and_simulated_annealing"><span class="tocnumber">6</span> <span class="toctext">Genetic algorithms and simulated annealing</span></a></li>
<li class="toclevel-1 tocsection-7"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Simulated_quantum_computing"><span class="tocnumber">7</span> <span class="toctext">Simulated quantum computing</span></a></li>
<li class="toclevel-1 tocsection-8"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Phylogeny-aware_methods"><span class="tocnumber">8</span> <span class="toctext">Phylogeny-aware methods</span></a></li>
<li class="toclevel-1 tocsection-9"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Motif_finding"><span class="tocnumber">9</span> <span class="toctext">Motif finding</span></a></li>
<li class="toclevel-1 tocsection-10"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Non-Coding_Multiple_Sequence_Alignment"><span class="tocnumber">10</span> <span class="toctext">Non-Coding Multiple Sequence Alignment</span></a></li>
<li class="toclevel-1 tocsection-11"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Alignment_visualization_and_quality_control"><span class="tocnumber">11</span> <span class="toctext">Alignment visualization and quality control</span></a></li>
<li class="toclevel-1 tocsection-12"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Use_in_phylogenetics"><span class="tocnumber">12</span> <span class="toctext">Use in phylogenetics</span></a></li>
<li class="toclevel-1 tocsection-13"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#See_also"><span class="tocnumber">13</span> <span class="toctext">See also</span></a></li>
<li class="toclevel-1 tocsection-14"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#References"><span class="tocnumber">14</span> <span class="toctext">References</span></a>
<ul>
<li class="toclevel-2 tocsection-15"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Survey_articles"><span class="tocnumber">14.1</span> <span class="toctext">Survey articles</span></a></li>
</ul>
</li>
<li class="toclevel-1 tocsection-16"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#External_links"><span class="tocnumber">15</span> <span class="toctext">External links</span></a>
<ul>
<li class="toclevel-2 tocsection-17"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#Lecture_notes.2C_tutorials.2C_and_courses"><span class="tocnumber">15.1</span> <span class="toctext">Lecture notes, tutorials, and courses</span></a></li>
</ul>
</li>
</ul>
</div>
<p></p>
<h2><span class="mw-headline" id="Dynamic_programming_and_computational_complexity">Dynamic programming and computational complexity</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=1" title="Edit section: Dynamic programming and computational complexity">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>A direct method for producing an MSA uses the <a href="https://en.wikipedia.org/wiki/Dynamic_programming" title="Dynamic programming">dynamic programming</a> technique to identify the globally optimal alignment solution. For proteins, this method usually involves two sets of parameters: a <a href="https://en.wikipedia.org/wiki/Gap_penalty" title="Gap penalty">gap penalty</a> and a <a href="https://en.wikipedia.org/wiki/Substitution_matrix" title="Substitution matrix">substitution matrix</a> assigning scores or probabilities to the alignment of each possible pair of amino acids based on the similarity of the amino acids' chemical properties and the evolutionary probability of the mutation. For nucleotide sequences a similar gap penalty is used, but a much simpler substitution matrix, wherein only identical matches and mismatches are considered, is typical. The scores in the substitution matrix may be either all positive or a mix of positive and negative in the case of a global alignment, but must be both positive and negative, in the case of a local alignment.<sup id="cite_ref-1" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-1">[1]</a></sup></p>
<p>For <i>n</i> individual sequences, the naive method requires constructing the <i>n</i>-dimensional equivalent of the matrix formed in standard pairwise <a href="https://en.wikipedia.org/wiki/Sequence_alignment" title="Sequence alignment">sequence alignment</a>. The search space thus increases exponentially with increasing <i>n</i> and is also strongly dependent on sequence length. Expressed with the <a href="https://en.wikipedia.org/wiki/Big_O_notation" title="Big O notation">big O notation</a> commonly used to measure <a href="https://en.wikipedia.org/wiki/Computational_complexity_theory" title="Computational complexity theory">computational complexity</a>, a <a href="https://en.wikipedia.org/wiki/Na%C3%AFve_algorithm" class="mw-redirect" title="Naïve algorithm">naïve</a> MSA takes <i>O(Length<sup>Nseqs</sup>)</i> time to produce. To find the global optimum for <i>n</i> sequences this way has been shown to be an <a href="https://en.wikipedia.org/wiki/NP_complete" class="mw-redirect" title="NP complete">NP-complete</a> problem.<sup id="cite_ref-wang_2-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-wang-2">[2]</a></sup><sup id="cite_ref-just_3-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-just-3">[3]</a></sup><sup id="cite_ref-elias_4-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-elias-4">[4]</a></sup> In 1989, based on Carrillo-Lipman Algorithm,<sup id="cite_ref-carrillo_5-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-carrillo-5">[5]</a></sup> Altschul introduced a practical method that uses pairwise alignments to constrain the n-dimensional search space.<sup id="cite_ref-altschul_6-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-altschul-6">[6]</a></sup> In this approach pairwise dynamic programming alignments are performed on each pair of sequences in the query set, and only the space near the n-dimensional intersection of these alignments is searched for the n-way alignment. The MSA program optimizes the sum of all of the pairs of characters at each position in the alignment (the so-called <i>sum of pair</i> score) and has been implemented in a software program for constructing multiple sequence alignments.<sup id="cite_ref-7" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-7">[7]</a></sup></p>
<h2><span class="mw-headline" id="Progressive_alignment_construction">Progressive alignment construction</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=2" title="Edit section: Progressive alignment construction">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>The most widely used approach to multiple sequence alignments uses a heuristic search known as progressive technique (also known as the hierarchical or tree method) developed by <a href="https://en.wikipedia.org/wiki/Paulien_Hogeweg" title="Paulien Hogeweg">Paulien Hogeweg</a> and Ben Hesper in 1984.<sup id="cite_ref-hogeweg1984_8-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-hogeweg1984-8">[8]</a></sup> Progressive alignment builds up a final MSA by combining pairwise alignments beginning with the most similar pair and progressing to the most distantly related. All progressive alignment methods require two stages: a first stage in which the relationships between the sequences are represented as a <a href="https://en.wikipedia.org/wiki/Phylogenetic_tree" title="Phylogenetic tree">tree</a>, called a <i>guide tree</i>, and a second step in which the MSA is built by adding the sequences sequentially to the growing MSA according to the guide tree. The initial <i>guide tree</i> is determined by an efficient <a href="https://en.wikipedia.org/wiki/Cluster_analysis" title="Cluster analysis">clustering</a> method such as <a href="https://en.wikipedia.org/wiki/Neighbor-joining" class="mw-redirect" title="Neighbor-joining">neighbor-joining</a> or <a href="https://en.wikipedia.org/wiki/UPGMA" title="UPGMA">UPGMA</a>, and may use distances based on the number of identical two letter sub-sequences (as in <a href="https://en.wikipedia.org/wiki/FASTA" title="FASTA">FASTA</a> rather than a dynamic programming alignment).<sup id="cite_ref-mount_9-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-mount-9">[9]</a></sup></p>
<p>Progressive alignments are not guaranteed to be globally optimal. The primary problem is that when errors are made at any stage in growing the MSA, these errors are then propagated through to the final result. Performance is also particularly bad when all of the sequences in the set are rather distantly related. Most modern progressive methods modify their scoring function with a secondary weighting function that assigns scaling factors to individual members of the query set in a nonlinear fashion based on their phylogenetic distance from their nearest neighbors. This corrects for non-random selection of the sequences given to the alignment program.<sup id="cite_ref-mount_9-1" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-mount-9">[9]</a></sup></p>
<p>Progressive alignment methods are efficient enough to implement on a large scale for many (100s to 1000s) sequences. Progressive alignment services are commonly available on publicly accessible web servers so users need not locally install the applications of interest. The most popular progressive alignment method has been the <a href="https://en.wikipedia.org/wiki/Clustal" title="Clustal">Clustal</a> family,<sup id="cite_ref-higgins1988_10-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-higgins1988-10">[10]</a></sup> especially the weighted variant ClustalW<sup id="cite_ref-thomson1994_11-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-thomson1994-11">[11]</a></sup> to which access is provided by a large number of web portals including <a rel="nofollow" class="external text" href="http://align.genome.jp/">GenomeNet</a>, <a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/Tools/clustalw2/index.html">EBI</a>, and <a rel="nofollow" class="external text" href="http://www.ch.embnet.org/software/ClustalW.html">EMBNet</a>. Different portals or implementations can vary in user interface and make different parameters accessible to the user. ClustalW is used extensively for phylogenetic tree construction, in spite of the author's explicit warnings that unedited alignments should not be used in such studies and as input for <a href="https://en.wikipedia.org/wiki/Protein_structure_prediction" title="Protein structure prediction">protein structure prediction</a> by homology modeling. Current version of Clustal family is ClustalW2. EMBL-EBI announced that CLustalW2 will be expired in August 2015. They recommend Clustal Omega which performs based on seeded guide trees and HMM profile-profile techniques for protein alignments. They offer different MSA tools for progressive DNA alignments. One of them is <a href="https://en.wikipedia.org/wiki/MAFFT" title="MAFFT">MAFFT</a> (Multiple Alignment using Fast Fourier Transform).<sup id="cite_ref-EMBL-EBI_12-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-EMBL-EBI-12">[12]</a></sup></p>
<p>Another common progressive alignment method called <a href="https://en.wikipedia.org/wiki/T-Coffee" title="T-Coffee">T-Coffee</a><sup id="cite_ref-notredame2000_13-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-notredame2000-13">[13]</a></sup> is slower than Clustal and its derivatives but generally produces more accurate alignments for distantly related sequence sets. T-Coffee calculates pairwise alignments by combining the direct alignment of the pair with indirect alignments that aligns each sequence of the pair to a third sequence. It uses the output from Clustal as well as another local alignment program LALIGN, which finds multiple regions of local alignment between two sequences. The resulting alignment and phylogenetic tree are used as a guide to produce new and more accurate weighting factors.</p>
<p>Because progressive methods are heuristics that are not guaranteed to converge to a global optimum, alignment quality can be difficult to evaluate and their true biological significance can be obscure. A semi-progressive method that improves alignment quality and does not use a lossy heuristic while still running in <a href="https://en.wikipedia.org/wiki/Polynomial_time" class="mw-redirect" title="Polynomial time">polynomial time</a> has been implemented in the program <a rel="nofollow" class="external text" href="http://faculty.cs.tamu.edu/shsze/psalign/">PSAlign</a>.<sup id="cite_ref-sze2006_14-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-sze2006-14">[14]</a></sup></p>
<h2><span class="mw-headline" id="Iterative_methods">Iterative methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=3" title="Edit section: Iterative methods">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>A set of methods to produce MSAs while reducing the errors inherent in progressive methods are classified as "iterative" because they work similarly to progressive methods but repeatedly realign the initial sequences as well as adding new sequences to the growing MSA. One reason progressive methods are so strongly dependent on a high-quality initial alignment is the fact that these alignments are always incorporated into the final result&nbsp;— that is, once a sequence has been aligned into the MSA, its alignment is not considered further. This approximation improves efficiency at the cost of accuracy. By contrast, iterative methods can return to previously calculated pairwise alignments or sub-MSAs incorporating subsets of the query sequence as a means of optimizing a general <a href="https://en.wikipedia.org/wiki/Objective_function" class="mw-redirect" title="Objective function">objective function</a> such as finding a high-quality alignment score.<sup id="cite_ref-mount_9-2" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-mount-9">[9]</a></sup></p>
<p>A variety of subtly different iteration methods have been implemented and made available in software packages; reviews and comparisons have been useful but generally refrain from choosing a "best" technique.<sup id="cite_ref-hirosawa_15-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-hirosawa-15">[15]</a></sup> The software package <a rel="nofollow" class="external text" href="http://www.genome.jp/tools/prrn/">PRRN/PRRP</a> uses a <a href="https://en.wikipedia.org/wiki/Hill-climbing_algorithm" class="mw-redirect" title="Hill-climbing algorithm">hill-climbing algorithm</a> to optimize its MSA alignment score<sup id="cite_ref-gotoh_16-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-gotoh-16">[16]</a></sup> and iteratively corrects both alignment weights and locally divergent or "gappy" regions of the growing MSA.<sup id="cite_ref-mount_9-3" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-mount-9">[9]</a></sup> PRRP performs best when refining an alignment previously constructed by a faster method.<sup id="cite_ref-mount_9-4" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-mount-9">[9]</a></sup></p>
<p>Another iterative program, DIALIGN, takes an unusual approach of focusing narrowly on local alignments between sub-segments or <a href="https://en.wikipedia.org/wiki/Sequence_motif" title="Sequence motif">sequence motifs</a> without introducing a gap penalty.<sup id="cite_ref-brudno_17-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-brudno-17">[17]</a></sup> The alignment of individual motifs is then achieved with a matrix representation similar to a dot-matrix plot in a pairwise alignment. An alternative method that uses fast local alignments as anchor points or "seeds" for a slower global-alignment procedure is implemented in the <a rel="nofollow" class="external text" href="http://dialign.gobics.de/chaos-dialign-submission">CHAOS/DIALIGN</a> suite.<sup id="cite_ref-brudno_17-1" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-brudno-17">[17]</a></sup></p>
<p>A third popular iteration-based method called <a href="https://en.wikipedia.org/wiki/MUSCLE_(alignment_software)" title="MUSCLE (alignment software)">MUSCLE</a> (multiple sequence alignment by log-expectation) improves on progressive methods with a more accurate distance measure to assess the relatedness of two sequences.<sup id="cite_ref-edgar_18-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-edgar-18">[18]</a></sup> The distance measure is updated between iteration stages (although, in its original form, MUSCLE contained only 2-3 iterations depending on whether refinement was enabled).</p>
<h2><span class="mw-headline" id="Consensus_methods">Consensus methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=4" title="Edit section: Consensus methods">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Consensus methods attempt to find the optimal multiple sequence alignment given multiple different alignments of the same set of sequences. There are two commonly used consensus methods, <a rel="nofollow" class="external text" href="http://www.tcoffee.org/Projects/mcoffee/">M-COFFEE</a> and <a rel="nofollow" class="external text" href="http://www.stevekellylab.com/software/mergealign">MergeAlign</a>.<sup id="cite_ref-mergealign_19-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-mergealign-19">[19]</a></sup> M-COFFEE uses multiple sequence alignments generated by seven different methods to generate consensus alignments. MergeAlign is capable of generating consensus alignments from any number of input alignments generated using different models of sequence evolution or different methods of multiple sequence alignment. The default option for MergeAlign is to infer a consensus alignment using alignments generated using 91 different models of protein sequence evolution.</p>
<h2><span class="mw-headline" id="Hidden_Markov_models">Hidden Markov models</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=5" title="Edit section: Hidden Markov models">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p><a href="https://en.wikipedia.org/wiki/Hidden_Markov_model" title="Hidden Markov model">Hidden Markov models</a> are probabilistic models that can assign likelihoods to all possible combinations of gaps, matches, and mismatches to determine the most likely MSA or set of possible MSAs. HMMs can produce a single highest-scoring output but can also generate a family of possible alignments that can then be evaluated for biological significance. HMMs can produce both global and local alignments. Although HMM-based methods have been developed relatively recently, they offer significant improvements in computational speed, especially for sequences that contain overlapping regions.<sup id="cite_ref-mount_9-5" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-mount-9">[9]</a></sup></p>
<p>Typical HMM-based methods work by representing an MSA as a form of <a href="https://en.wikipedia.org/wiki/Directed_acyclic_graph" title="Directed acyclic graph">directed acyclic graph</a> known as a partial-order graph, which consists of a series of nodes representing possible entries in the columns of an MSA. In this representation a column that is absolutely conserved (that is, that all the sequences in the MSA share a particular character at a particular position) is coded as a single node with as many outgoing connections as there are possible characters in the next column of the alignment. In the terms of a typical hidden Markov model, the observed states are the individual alignment columns and the "hidden" states represent the presumed ancestral sequence from which the sequences in the query set are hypothesized to have descended. An efficient search variant of the dynamic programming method, known as the <a href="https://en.wikipedia.org/wiki/Viterbi_algorithm" title="Viterbi algorithm">Viterbi algorithm</a>, is generally used to successively align the growing MSA to the next sequence in the query set to produce a new MSA.<sup id="cite_ref-hughey_20-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-hughey-20">[20]</a></sup> This is distinct from progressive alignment methods because the alignment of prior sequences is updated at each new sequence addition. However, like progressive methods, this technique can be influenced by the order in which the sequences in the query set are integrated into the alignment, especially when the sequences are distantly related.<sup id="cite_ref-mount_9-6" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-mount-9">[9]</a></sup></p>
<p>Several software programs are available in which variants of HMM-based methods have been implemented and which are noted for their scalability and efficiency, although properly using an HMM method is more complex than using more common progressive methods. The simplest is <a rel="nofollow" class="external text" href="http://sourceforge.net/projects/poamsa/files/">POA</a> (Partial-Order Alignment);<sup id="cite_ref-grasso_21-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-grasso-21">[21]</a></sup> a similar but more generalized method is implemented in the packages <a rel="nofollow" class="external text" href="http://compbio.soe.ucsc.edu/sam.html">SAM</a> (Sequence Alignment and Modeling System).<sup id="cite_ref-hugheyT_22-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-hugheyT-22">[22]</a></sup> and <a href="https://en.wikipedia.org/wiki/HMMER" title="HMMER">HMMER</a>.<sup id="cite_ref-durbin_23-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-durbin-23">[23]</a></sup> SAM has been used as a source of alignments for <a href="https://en.wikipedia.org/wiki/Protein_structure_prediction" title="Protein structure prediction">protein structure prediction</a> to participate in the <a href="https://en.wikipedia.org/wiki/CASP" title="CASP">CASP</a> structure prediction experiment and to develop a database of predicted proteins in the <a href="https://en.wikipedia.org/wiki/Yeast" title="Yeast">yeast</a> species <i><a href="https://en.wikipedia.org/wiki/S._cerevisiae" class="mw-redirect" title="S. cerevisiae">S. cerevisiae</a></i>. <a href="https://en.wikipedia.org/wiki/HHpred_/_HHsearch" title="HHpred / HHsearch">HHsearch</a><sup id="cite_ref-24" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-24">[24]</a></sup> is a software package for the detection of remotely related protein sequences based on the pairwise comparison of HMMs. A server running HHsearch (<a href="https://en.wikipedia.org/wiki/HHpred_/_HHsearch" title="HHpred / HHsearch">HHpred</a>) was by far the fastest of the 10 best automatic structure prediction servers in the CASP7 and CASP8 structure prediction competitions.<sup id="cite_ref-25" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-25">[25]</a></sup></p>
<h2><span class="mw-headline" id="Genetic_algorithms_and_simulated_annealing">Genetic algorithms and simulated annealing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=6" title="Edit section: Genetic algorithms and simulated annealing">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Standard optimization techniques in computer science&nbsp;— both of which were inspired by, but do not directly reproduce, physical processes&nbsp;— have also been used in an attempt to more efficiently produce quality MSAs. One such technique, <a href="https://en.wikipedia.org/wiki/Genetic_algorithm" title="Genetic algorithm">genetic algorithms</a>, has been used for MSA production in an attempt to broadly simulate the hypothesized evolutionary process that gave rise to the divergence in the query set. The method works by breaking a series of possible MSAs into fragments and repeatedly rearranging those fragments with the introduction of gaps at varying positions. A general <a href="https://en.wikipedia.org/wiki/Objective_function" class="mw-redirect" title="Objective function">objective function</a> is optimized during the simulation, most generally the "sum of pairs" maximization function introduced in dynamic programming-based MSA methods. A technique for protein sequences has been implemented in the software program SAGA (Sequence Alignment by Genetic Algorithm)<sup id="cite_ref-notredame2_26-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-notredame2-26">[26]</a></sup> and its equivalent in RNA is called RAGA.<sup id="cite_ref-notredame3_27-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-notredame3-27">[27]</a></sup></p>
<p>The technique of <a href="https://en.wikipedia.org/wiki/Simulated_annealing" title="Simulated annealing">simulated annealing</a>, by which an existing MSA produced by another method is refined by a series of rearrangements designed to find better regions of alignment space than the one the input alignment already occupies. Like the genetic algorithm method, simulated annealing maximizes an objective function like the sum-of-pairs function. Simulated annealing uses a metaphorical "temperature factor" that determines the rate at which rearrangements proceed and the likelihood of each rearrangement; typical usage alternates periods of high rearrangement rates with relatively low likelihood (to explore more distant regions of alignment space) with periods of lower rates and higher likelihoods to more thoroughly explore local minima near the newly "colonized" regions. This approach has been implemented in the program MSASA (Multiple Sequence Alignment by Simulated Annealing).<sup id="cite_ref-kim_28-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-kim-28">[28]</a></sup></p>
<h2><span class="mw-headline" id="Simulated_quantum_computing">Simulated quantum computing</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=7" title="Edit section: Simulated quantum computing">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>In January 2017, <a href="https://en.wikipedia.org/wiki/D-Wave_Systems" title="D-Wave Systems">D-Wave Systems</a> announced that its qbsolv open-source quantum computing software had been successfully used to find a faster solution to the MSA problem.<sup id="cite_ref-29" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-29">[29]</a></sup></p>
<h2><span class="mw-headline" id="Phylogeny-aware_methods">Phylogeny-aware methods</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=8" title="Edit section: Phylogeny-aware methods">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="thumb tright">
<div class="thumbinner" style="width:452px;"><a href="https://en.wikipedia.org/wiki/File:Non-homologous_exon_alignment.jpg" class="image"><img alt="" src="./Multiple sequence alignment - Wikipedia_files/450px-Non-homologous_exon_alignment.jpg" width="450" height="144" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/e/e4/Non-homologous_exon_alignment.jpg/675px-Non-homologous_exon_alignment.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/e/e4/Non-homologous_exon_alignment.jpg 2x" data-file-width="800" data-file-height="256"></a>
<div class="thumbcaption">
<div class="magnify"><a href="https://en.wikipedia.org/wiki/File:Non-homologous_exon_alignment.jpg" class="internal" title="Enlarge"></a></div>
Non-homologous exon alignment by an iterative method (a), and by a phylogeny-aware method (b)</div>
</div>
</div>
<p>Most multiple sequence alignment methods try to minimize the number of <a href="https://en.wikipedia.org/wiki/Indel" title="Indel">insertions/deletions</a> (gaps) and, as a consequence, produce compact alignments. This causes several problems if the sequences to be aligned contain non-<a href="https://en.wikipedia.org/wiki/Homology_(biology)#Sequence_homology" title="Homology (biology)">homologous</a> regions, if gaps are informative in a <a href="https://en.wikipedia.org/wiki/Phylogeny" class="mw-redirect" title="Phylogeny">phylogeny</a> analysis. These problems are common in newly produced sequences that are poorly annotated and may contain <a href="https://en.wikipedia.org/wiki/Frameshift" class="mw-redirect" title="Frameshift">frame-shifts</a>, wrong <a href="https://en.wikipedia.org/wiki/Protein_domain" title="Protein domain">domains</a> or non-homologous <a href="https://en.wikipedia.org/wiki/RNA_splicing" title="RNA splicing">spliced</a> <a href="https://en.wikipedia.org/wiki/Exons" class="mw-redirect" title="Exons">exons</a>.</p>
<p>The first such method was developed in 2005 by Löytynoja and Goldman.<sup id="cite_ref-Loytynoja-2005_30-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-Loytynoja-2005-30">[30]</a></sup> The same authors released a software package called <i>PRANK</i> in 2008.<sup id="cite_ref-Loytynoja-2008_31-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-Loytynoja-2008-31">[31]</a></sup> PRANK improves alignments when insertions are present. Nevertheless, it runs slowly compared to progressive and/or iterative methods which have been developed for several years.</p>
<p>In 2012, two new phylogeny-aware tools appeared. One is called <i>PAGAN</i> that was developed by the same team as PRANK.<sup id="cite_ref-Loytynoja-2012_32-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-Loytynoja-2012-32">[32]</a></sup> The other is <i>ProGraphMSA</i> developed by Szalkowski.<sup id="cite_ref-Szalkowski-2012_33-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-Szalkowski-2012-33">[33]</a></sup> Both software packages were developed independently but share common features, notably the use of <a href="https://en.wikipedia.org/wiki/Directed_acyclic_graph#Partial_orders_and_topological_ordering" title="Directed acyclic graph">graph algorithms</a> to improve the recognition of non-homologous regions, and an improvement in code making these software faster than PRANK.</p>
<h2><span class="mw-headline" id="Motif_finding">Motif finding</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=9" title="Edit section: Motif finding">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="thumb tright">
<div class="thumbinner" style="width:502px;"><a href="https://en.wikipedia.org/wiki/File:Caspase-motif-alignment.png" class="image"><img alt="" src="./Multiple sequence alignment - Wikipedia_files/500px-Caspase-motif-alignment.png" width="500" height="82" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/9/94/Caspase-motif-alignment.png/750px-Caspase-motif-alignment.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/9/94/Caspase-motif-alignment.png/1000px-Caspase-motif-alignment.png 2x" data-file-width="1070" data-file-height="175"></a>
<div class="thumbcaption">
<div class="magnify"><a href="https://en.wikipedia.org/wiki/File:Caspase-motif-alignment.png" class="internal" title="Enlarge"></a></div>
Alignment of the seven <a href="https://en.wikipedia.org/wiki/Drosophila" title="Drosophila">Drosophila</a> <a href="https://en.wikipedia.org/wiki/Caspase" title="Caspase">caspases</a> colored by motifs as identified by MEME. When motif positions and sequence alignments are generated independently, they often correlate well but not perfectly, as in this example.</div>
</div>
</div>
<p>Motif finding, also known as profile analysis, is a method of locating <a href="https://en.wikipedia.org/wiki/Sequence_motif" title="Sequence motif">sequence motifs</a> in global MSAs that is both a means of producing a better MSA and a means of producing a scoring matrix for use in searching other sequences for similar motifs. A variety of methods for isolating the motifs have been developed, but all are based on identifying short highly conserved patterns within the larger alignment and constructing a matrix similar to a substitution matrix that reflects the amino acid or nucleotide composition of each position in the putative motif. The alignment can then be refined using these matrices. In standard profile analysis, the matrix includes entries for each possible character as well as entries for gaps.<sup id="cite_ref-mount_9-7" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-mount-9">[9]</a></sup> Alternatively, statistical pattern-finding algorithms can identify motifs as a precursor to an MSA rather than as a derivation. In many cases when the query set contains only a small number of sequences or contains only highly related sequences, <a href="https://en.wikipedia.org/wiki/Pseudocount" class="mw-redirect" title="Pseudocount">pseudocounts</a> are added to normalize the distribution reflected in the scoring matrix. In particular, this corrects zero-probability entries in the matrix to values that are small but nonzero.</p>
<p>Blocks analysis is a method of motif finding that restricts motifs to ungapped regions in the alignment. Blocks can be generated from an MSA or they can be extracted from unaligned sequences using a precalculated set of common motifs previously generated from known gene families.<sup id="cite_ref-henikoff1991_34-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-henikoff1991-34">[34]</a></sup> Block scoring generally relies on the spacing of high-frequency characters rather than on the calculation of an explicit substitution matrix. The <a rel="nofollow" class="external text" href="http://blocks.fhcrc.org/">BLOCKS</a> server provides an interactive method to locate such motifs in unaligned sequences.</p>
<p>Statistical pattern-matching has been implemented using both the <a href="https://en.wikipedia.org/wiki/Expectation-maximization_algorithm" class="mw-redirect" title="Expectation-maximization algorithm">expectation-maximization algorithm</a> and the <a href="https://en.wikipedia.org/wiki/Gibbs_sampler" class="mw-redirect" title="Gibbs sampler">Gibbs sampler</a>. One of the most common motif-finding tools, known as <a href="https://en.wikipedia.org/wiki/Multiple_EM_for_Motif_Elicitation" title="Multiple EM for Motif Elicitation">MEME</a>, uses expectation maximization and hidden Markov methods to generate motifs that are then used as search tools by its companion MAST in the combined suite <a rel="nofollow" class="external text" href="http://meme.sdsc.edu/meme/intro.html">MEME/MAST</a>.<sup id="cite_ref-baileyelkan1994_35-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-baileyelkan1994-35">[35]</a></sup><sup id="cite_ref-baileygribskov1998_36-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-baileygribskov1998-36">[36]</a></sup></p>
<h2><span class="mw-headline" id="Non-Coding_Multiple_Sequence_Alignment">Non-Coding Multiple Sequence Alignment</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=10" title="Edit section: Non-Coding Multiple Sequence Alignment">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Non-coding DNA regions, especially TFBSs, are rather more conserved and not necessarily evolutionarily related, and may have converged from non-common ancestors. Thus, the assumptions used to align protein sequences and DNA coding regions are inherently different from those that hold for TFBS sequences. Although it is meaningful to align DNA coding regions for homologous sequences using mutation operators, alignment of binding site sequences for the same transcription factor cannot rely on evolutionary related mutation operations. Similarly, the evolutionary operator of point mutations can be used to define an edit distance for coding sequences, but this has little meaning for TFBS sequences because any sequence variation has to maintain a certain level of specificity for the binding site to function. This becomes specifically important when trying to align known TFBS sequences to build supervised models to predict unknown locations of the same TFBS. Hence, Multiple Sequence Alignment methods need to adjust the underlying evolutionary hypothesis and the operators used as in the work published incorporating neighbouring base thermodynamic information <sup id="cite_ref-Salama2013_37-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-Salama2013-37">[37]</a></sup> to align the binding sites searching for the lowest thermodynamic alignment conserving specificity of the binding site, <a rel="nofollow" class="external text" href="http://sourceforge.net/projects/msa-edna/">EDNA</a> .</p>
<h2><span class="mw-headline" id="Alignment_visualization_and_quality_control">Alignment visualization and quality control</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=11" title="Edit section: Alignment visualization and quality control">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>The necessary use of heuristics for multiple alignment means that for an arbitrary set of proteins, there is always a good chance that an alignment will contain errors. For example, an evaluation of several leading alignment programs using the <a href="https://en.wikipedia.org/wiki/List_of_sequence_alignment_software#Benchmarking" title="List of sequence alignment software">BAliBase benchmark</a> found that at least 24% of all pairs of aligned amino acids were incorrectly aligned.<sup id="cite_ref-nuin2006_38-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-nuin2006-38">[38]</a></sup> These errors can arise because of unique insertions into one or more regions of sequences, or through some more complex evolutionary process leading to proteins that do not align easily by sequence alone. As the number of sequence and their divergence increases many more errors will be made simply because of the heuristic nature of MSA algorithms. <a href="https://en.wikipedia.org/wiki/List_of_alignment_visualization_software" title="List of alignment visualization software">Multiple sequence alignment viewers</a> enable alignments to be visually reviewed, often by inspecting the quality of alignment for annotated functional sites on two or more sequences. Many also enable the alignment to be edited to correct these (usually minor) errors, in order to obtain an optimal 'curated' alignment suitable for use in phylogenetic analysis or comparative modeling.<sup id="cite_ref-39" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-39">[39]</a></sup></p>
<p>However, as the number of sequences increases and especially in genome-wide studies that involve many MSAs it is impossible to manually curate all alignments. Furthermore, manual curation is subjective. And finally, even the best expert cannot confidently align the more ambiguous cases of highly diverged sequences. In such cases it is common practice to use automatic procedures to exclude unreliably aligned regions from the MSA. For the purpose of phylogeny reconstruction (see below) the Gblocks program is widely used to remove alignment blocks suspect of low quality, according to various cutoffs on the number of gapped sequences in alignment columns.<sup id="cite_ref-castresana2000_40-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-castresana2000-40">[40]</a></sup> However, these criteria may excessively filter out regions with insertion/deletion events that may still be aligned reliably, and these regions might be desirable for other purposes such as detection of positive selection. A few alignment algorithms output site-specific scores that allow the selection of high-confidence regions. Such a service was first offered by the SOAP program,<sup id="cite_ref-loytynojaMilinkovitch2001_41-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-loytynojaMilinkovitch2001-41">[41]</a></sup> which tests the robustness of each column to perturbation in the parameters of the popular alignment program CLUSTALW. The T-Coffee program<sup id="cite_ref-poirotOTooleNotredame2003_42-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-poirotOTooleNotredame2003-42">[42]</a></sup> uses a library of alignments in the construction of the final MSA, and its output MSA is colored according to confidence scores that reflect the agreement between different alignments in the library regarding each aligned residue. Its extension, <a rel="nofollow" class="external text" href="http://tcoffee.crg.cat/tcs">TCS</a>&nbsp;: (<b>T</b>ransitive <b>C</b>onsistency <b>S</b>core), uses T-Coffee libraries of pairwise alignments to evaluate any third party MSA. Pairwise projections can be produced using fast or slow methods, thus allowing a trade-off between speed and accuracy.<sup id="cite_ref-TCS2014MBE_43-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-TCS2014MBE-43">[43]</a></sup><sup id="cite_ref-TCS_2015_NAR_44-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-TCS_2015_NAR-44">[44]</a></sup> Another alignment program that can output an MSA with confidence scores is FSA,<sup id="cite_ref-bradley2009_45-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-bradley2009-45">[45]</a></sup> which uses a statistical model that allows calculation of the uncertainty in the alignment. The HoT (Heads-Or-Tails) score can be used as a measure of site-specific alignment uncertainty due to the existence of multiple co-optimal solutions.<sup id="cite_ref-landanGraur2008_46-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-landanGraur2008-46">[46]</a></sup> The GUIDANCE program<sup id="cite_ref-penn2010_47-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-penn2010-47">[47]</a></sup> calculates a similar site-specific confidence measure based on the robustness of the alignment to uncertainty in the guide tree that is used in progressive alignment programs. An alternative, more statistically justified approach to assess alignment uncertainty is the use of probabilistic evolutionary models for joint estimation of phylogeny and alignment. A Bayesian approach allows calculation of posterior probabilities of estimated phylogeny and alignment, which is a measure of the confidence in these estimates. In this case, a posterior probability can be calculated for each site in the alignment. Such an approach was implemented in the program BAli-Phy.<sup id="cite_ref-redelingsSuchard2005_48-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-redelingsSuchard2005-48">[48]</a></sup></p>
<p>There free available programs for visualization of multiple sequence alignments: <a href="https://en.wikipedia.org/w/index.php?title=JalView&amp;action=edit&amp;redlink=1" class="new" title="JalView (page does not exist)">JalView</a>, <a href="https://en.wikipedia.org/wiki/UGENE" title="UGENE">UGENE</a>.</p>
<h2><span class="mw-headline" id="Use_in_phylogenetics">Use in phylogenetics</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=12" title="Edit section: Use in phylogenetics">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<p>Multiple sequence alignments can be used to create a <a href="https://en.wikipedia.org/wiki/Phylogenetic_tree" title="Phylogenetic tree">phylogenetic tree</a>.<sup id="cite_ref-Budd_2009_49-0" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-Budd_2009-49">[49]</a></sup> This is made possible by two reasons. The first is because functional domains that are known in annotated sequences can be used for alignment in non-annotated sequences. The other is that conserved regions known to be functionally important can be found. This makes it possible for multiple sequence alignments to be used to analyze and find evolutionary relationships through homology between sequences. Point mutations and insertion or deletion events (called indels) can be detected.</p>
<p>Multiple sequence alignments can also be used to identify functionally important sites, such as binding sites, active sites, or sites corresponding to other key functions, by locating conserved domains. When looking at multiple sequence alignments, it is useful to consider different aspects of the sequences when comparing sequences. These aspects include identity, similarity, and homology. Identity means that the sequences have identical residues at their respective positions. On the other hand, similarity has to do with the sequences being compared having similar residues quantitatively. For example, in terms of nucleotide sequences, pyrimidines are considered similar to each other, as are purines. Similarity ultimately leads to homology, in that the more similar sequences are, the closer they are to being homologous. This similarity in sequences can then go on to help find common ancestry.<sup id="cite_ref-Budd_2009_49-1" class="reference"><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_note-Budd_2009-49">[49]</a></sup></p>
<h2><span class="mw-headline" id="See_also">See also</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=13" title="Edit section: See also">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<ul>
<li><a href="https://en.wikipedia.org/wiki/Cladistics" title="Cladistics">Cladistics</a></li>
<li><a href="https://en.wikipedia.org/wiki/Generalized_tree_alignment" title="Generalized tree alignment">Generalized tree alignment</a></li>
<li><a href="https://en.wikipedia.org/wiki/Phylogenetics" title="Phylogenetics">Phylogenetics</a></li>
<li><a href="https://en.wikipedia.org/wiki/Sequence_alignment_software" class="mw-redirect" title="Sequence alignment software">Sequence alignment software</a></li>
<li><a href="https://en.wikipedia.org/wiki/List_of_alignment_visualization_software" title="List of alignment visualization software">Multiple Sequence Alignment viewers</a></li>
<li><a href="https://en.wikipedia.org/wiki/Structural_alignment" title="Structural alignment">Structural alignment</a></li>
<li><a href="https://en.wikipedia.org/wiki/Alignment-free_sequence_analysis" title="Alignment-free sequence analysis">Alignment-free sequence analysis</a></li>
</ul>
<h2><span class="mw-headline" id="References">References</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=14" title="Edit section: References">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<div class="reflist columns references-column-width" style="-moz-column-width: 30em; -webkit-column-width: 30em; column-width: 30em; list-style-type: decimal;">
<ol class="references">
<li id="cite_note-1"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-1"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/help/matrix.html">"Help with matrices used in sequence comparison tools"</a>. European Bioinformatics Institute<span class="reference-accessdate">. Retrieved <span class="nowrap">March 3,</span> 2010</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.btitle=Help+with+matrices+used+in+sequence+comparison+tools&amp;rft.genre=unknown&amp;rft.pub=European+Bioinformatics+Institute&amp;rft_id=http%3A%2F%2Fwww.ebi.ac.uk%2Fhelp%2Fmatrix.html&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-wang-2"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-wang_2-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Wang L, Jiang T (1994). "On the complexity of multiple sequence alignment". <i>J Comput Biol</i>. <b>1</b> (4): 337–348. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/8790475">8790475</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1089%2Fcmb.1994.1.337">10.1089/cmb.1994.1.337</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=On+the+complexity+of+multiple+sequence+alignment&amp;rft.au=Jiang%2C+T&amp;rft.aufirst=L&amp;rft.aulast=Wang&amp;rft.date=1994&amp;rft.genre=article&amp;rft.issue=4&amp;rft.jtitle=J+Comput+Biol&amp;rft.pages=337-348&amp;rft.volume=1&amp;rft_id=info%3Adoi%2F10.1089%2Fcmb.1994.1.337&amp;rft_id=info%3Apmid%2F8790475&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-just-3"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-just_3-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Just W (2001). "Computational complexity of multiple sequence alignment with SP-score". <i>J Comput Biol</i>. <b>8</b> (6): 615–23. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/11747615">11747615</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1089%2F106652701753307511">10.1089/106652701753307511</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Computational+complexity+of+multiple+sequence+alignment+with+SP-score&amp;rft.au=Just+W&amp;rft.date=2001&amp;rft.genre=article&amp;rft.issue=6&amp;rft.jtitle=J+Comput+Biol&amp;rft.pages=615-23&amp;rft.volume=8&amp;rft_id=info%3Adoi%2F10.1089%2F106652701753307511&amp;rft_id=info%3Apmid%2F11747615&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-elias-4"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-elias_4-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Elias, Isaac (2006). <a rel="nofollow" class="external text" href="http://www.liebertonline.com/doi/abs/10.1089/cmb.2006.13.1323">"Settling the intractability of multiple alignment"</a>. <i>J Comput Biol</i>. <b>13</b> (7): 1323–1339. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/17037961">17037961</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1089%2Fcmb.2006.13.1323">10.1089/cmb.2006.13.1323</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Settling+the+intractability+of+multiple+alignment&amp;rft.au=Elias%2C+Isaac&amp;rft.date=2006&amp;rft.genre=article&amp;rft.issue=7&amp;rft.jtitle=J+Comput+Biol&amp;rft.pages=1323-1339&amp;rft.volume=13&amp;rft_id=http%3A%2F%2Fwww.liebertonline.com%2Fdoi%2Fabs%2F10.1089%2Fcmb.2006.13.1323&amp;rft_id=info%3Adoi%2F10.1089%2Fcmb.2006.13.1323&amp;rft_id=info%3Apmid%2F17037961&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-carrillo-5"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-carrillo_5-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Carrillo H, Lipman DJ (1988). "The Multiple Sequence Alignment Problem in Biology". <i>SIAM Journal of Applied Mathematics</i>. <b>48</b> (5): 1073–1082. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1137%2F0148063">10.1137/0148063</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=The+Multiple+Sequence+Alignment+Problem+in+Biology&amp;rft.au=Lipman%2C+DJ&amp;rft.aufirst=H&amp;rft.aulast=Carrillo&amp;rft.date=1988&amp;rft.genre=article&amp;rft.issue=5&amp;rft.jtitle=SIAM+Journal+of+Applied+Mathematics&amp;rft.pages=1073-1082&amp;rft.volume=48&amp;rft_id=info%3Adoi%2F10.1137%2F0148063&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-altschul-6"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-altschul_6-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Lipman DJ, Altschul SF, Kececioglu JD (1989). <a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC287279">"A tool for multiple sequence alignment"</a>. <i>Proc Natl Acad Sci U S A</i>. <b>86</b> (12): 4412–4415. <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC287279">287279</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/2734293">2734293</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.86.12.4412">10.1073/pnas.86.12.4412</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=A+tool+for+multiple+sequence+alignment&amp;rft.au=Altschul%2C+SF&amp;rft.au=Kececioglu%2C+JD&amp;rft.aufirst=DJ&amp;rft.aulast=Lipman&amp;rft.date=1989&amp;rft.genre=article&amp;rft.issue=12&amp;rft.jtitle=Proc+Natl+Acad+Sci+U+S+A&amp;rft.pages=4412-4415&amp;rft.volume=86&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC287279&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC287279&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.86.12.4412&amp;rft_id=info%3Apmid%2F2734293&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-7"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-7"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://www.ncbi.nlm.nih.gov/CBBresearch/Schaffer/msa.html">"Genetic analysis software"</a>. National Center for Biotechnology Information<span class="reference-accessdate">. Retrieved <span class="nowrap">March 3,</span> 2010</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.btitle=Genetic+analysis+software&amp;rft.genre=unknown&amp;rft.pub=National+Center+for+Biotechnology+Information&amp;rft_id=http%3A%2F%2Fwww.ncbi.nlm.nih.gov%2FCBBresearch%2FSchaffer%2Fmsa.html&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-hogeweg1984-8"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-hogeweg1984_8-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Hogeweg P, Hesper B (1984). "The alignment of sets of sequences and the construction of phyletic trees: an integrated method". <i>J Mol Evol</i>. <b>20</b>: 1750186. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/6433036">6433036</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=The+alignment+of+sets+of+sequences+and+the+construction+of+phyletic+trees%3A+an+integrated+method&amp;rft.au=Hesper%2C+B&amp;rft.aufirst=P&amp;rft.aulast=Hogeweg&amp;rft.date=1984&amp;rft.genre=article&amp;rft.jtitle=J+Mol+Evol&amp;rft.pages=1750186&amp;rft.volume=20&amp;rft_id=info%3Apmid%2F6433036&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-mount-9"><span class="mw-cite-backlink">^ <a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-mount_9-0"><span class="cite-accessibility-label">Jump up to: </span><sup><i><b>a</b></i></sup></a> <a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-mount_9-1"><sup><i><b>b</b></i></sup></a> <a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-mount_9-2"><sup><i><b>c</b></i></sup></a> <a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-mount_9-3"><sup><i><b>d</b></i></sup></a> <a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-mount_9-4"><sup><i><b>e</b></i></sup></a> <a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-mount_9-5"><sup><i><b>f</b></i></sup></a> <a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-mount_9-6"><sup><i><b>g</b></i></sup></a> <a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-mount_9-7"><sup><i><b>h</b></i></sup></a></span> <span class="reference-text">Mount DM. (2004). Bioinformatics: Sequence and Genome Analysis 2nd ed. Cold Spring Harbor Laboratory Press: Cold Spring Harbor, NY.</span></li>
<li id="cite_note-higgins1988-10"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-higgins1988_10-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal"><a href="https://en.wikipedia.org/wiki/Desmond_G._Higgins" title="Desmond G. Higgins">Higgins DG</a>, Sharp PM (1988). "CLUSTAL: a package for performing multiple sequence alignment on a microcomputer". <i>Gene</i>. <b>73</b> (1): 237–244. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/3243435">3243435</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2F0378-1119%2888%2990330-7">10.1016/0378-1119(88)90330-7</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=CLUSTAL%3A+a+package+for+performing+multiple+sequence+alignment+on+a+microcomputer&amp;rft.au=Higgins+DG%2C+Sharp+PM&amp;rft.date=1988&amp;rft.genre=article&amp;rft.issue=1&amp;rft.jtitle=Gene&amp;rft.pages=237-244&amp;rft.volume=73&amp;rft_id=info%3Adoi%2F10.1016%2F0378-1119%2888%2990330-7&amp;rft_id=info%3Apmid%2F3243435&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-thomson1994-11"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-thomson1994_11-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Thompson JD, <a href="https://en.wikipedia.org/wiki/Desmond_G._Higgins" title="Desmond G. Higgins">Higgins DG</a>, Gibson TJ (1994). <a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC308517">"CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice"</a>. <i>Nucleic Acids Res</i>. <b>22</b> (22): 4673–4680. <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC308517">308517</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/7984417">7984417</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2F22.22.4673">10.1093/nar/22.22.4673</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=CLUSTAL+W%3A+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position-specific+gap+penalties+and+weight+matrix+choice&amp;rft.au=Thompson+JD%2C+Higgins+DG%2C+Gibson+TJ&amp;rft.date=1994&amp;rft.genre=article&amp;rft.issue=22&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.pages=4673-4680&amp;rft.volume=22&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC308517&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC308517&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F22.22.4673&amp;rft_id=info%3Apmid%2F7984417&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="https://en.wikipedia.org/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-EMBL-EBI-12"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-EMBL-EBI_12-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/Tools/msa/clustalw2/">"EMBL-EBI-ClustalW2-Multiple Sequence Alignment"</a>. <i>CLUSTALW2</i>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=EMBL-EBI-ClustalW2-Multiple+Sequence+Alignment&amp;rft.genre=unknown&amp;rft.jtitle=CLUSTALW2&amp;rft_id=http%3A%2F%2Fwww.ebi.ac.uk%2FTools%2Fmsa%2Fclustalw2%2F&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-notredame2000-13"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-notredame2000_13-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Notredame C, <a href="https://en.wikipedia.org/wiki/Desmond_G._Higgins" title="Desmond G. Higgins">Higgins DG</a>, Heringa J (2000). "T-Coffee: A novel method for fast and accurate multiple sequence alignment". <i>J Mol Biol</i>. <b>302</b> (1): 205–217. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/10964570">10964570</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1006%2Fjmbi.2000.4042">10.1006/jmbi.2000.4042</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=T-Coffee%3A+A+novel+method+for+fast+and+accurate+multiple+sequence+alignment&amp;rft.au=Notredame+C%2C+Higgins+DG%2C+Heringa+J&amp;rft.date=2000&amp;rft.genre=article&amp;rft.issue=1&amp;rft.jtitle=J+Mol+Biol&amp;rft.pages=205-217&amp;rft.volume=302&amp;rft_id=info%3Adoi%2F10.1006%2Fjmbi.2000.4042&amp;rft_id=info%3Apmid%2F10964570&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="https://en.wikipedia.org/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-sze2006-14"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-sze2006_14-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Sze SH, Lu Y, Yang Q (2006). "A polynomial time solvable formulation of multiple sequence alignment". <i>J Comput Biol</i>. <b>13</b> (2): 309–319. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/16597242">16597242</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1089%2Fcmb.2006.13.309">10.1089/cmb.2006.13.309</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=A+polynomial+time+solvable+formulation+of+multiple+sequence+alignment&amp;rft.au=Lu%2C+Y&amp;rft.au=Yang%2C+Q&amp;rft.aufirst=SH&amp;rft.aulast=Sze&amp;rft.date=2006&amp;rft.genre=article&amp;rft.issue=2&amp;rft.jtitle=J+Comput+Biol&amp;rft.pages=309-319&amp;rft.volume=13&amp;rft_id=info%3Adoi%2F10.1089%2Fcmb.2006.13.309&amp;rft_id=info%3Apmid%2F16597242&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-hirosawa-15"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-hirosawa_15-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Hirosawa M, Totoki Y, Hoshida M, Ishikawa M (1995). "Comprehensive study on iterative algorithms of multiple sequence alignment". <i>Comput Appl Biosci</i>. <b>11</b> (1): 13–18. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/7796270">7796270</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2F11.1.13">10.1093/bioinformatics/11.1.13</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Comprehensive+study+on+iterative+algorithms+of+multiple+sequence+alignment&amp;rft.au=Hoshida%2C+M&amp;rft.au=Ishikawa%2C+M&amp;rft.au=Totoki%2C+Y&amp;rft.aufirst=M&amp;rft.aulast=Hirosawa&amp;rft.date=1995&amp;rft.genre=article&amp;rft.issue=1&amp;rft.jtitle=Comput+Appl+Biosci&amp;rft.pages=13-18&amp;rft.volume=11&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2F11.1.13&amp;rft_id=info%3Apmid%2F7796270&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-gotoh-16"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-gotoh_16-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Gotoh O (1996). "Significant improvement in accuracy of multiple protein sequence alignments by iterative refinement as assessed by reference to structural alignments". <i>J Mol Biol</i>. <b>264</b> (4): 823–38. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/8980688">8980688</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1006%2Fjmbi.1996.0679">10.1006/jmbi.1996.0679</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Significant+improvement+in+accuracy+of+multiple+protein+sequence+alignments+by+iterative+refinement+as+assessed+by+reference+to+structural+alignments&amp;rft.au=Gotoh+O&amp;rft.date=1996&amp;rft.genre=article&amp;rft.issue=4&amp;rft.jtitle=J+Mol+Biol&amp;rft.pages=823-38&amp;rft.volume=264&amp;rft_id=info%3Adoi%2F10.1006%2Fjmbi.1996.0679&amp;rft_id=info%3Apmid%2F8980688&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-brudno-17"><span class="mw-cite-backlink">^ <a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-brudno_17-0"><span class="cite-accessibility-label">Jump up to: </span><sup><i><b>a</b></i></sup></a> <a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-brudno_17-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation journal">Brudno M, Chapman M, Göttgens B, Batzoglou S, Morgenstern B (2003). "Fast and sensitive multiple alignment of large genomic sequences". <i>BMC Bioinformatics</i>. <b>4</b>: 66. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-4-66">10.1186/1471-2105-4-66</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Fast+and+sensitive+multiple+alignment+of+large+genomic+sequences&amp;rft.au=Batzoglou%2C+S&amp;rft.au=Chapman%2C+M&amp;rft.au=G%C3%B6ttgens%2C+B&amp;rft.au=Morgenstern%2C+B&amp;rft.aufirst=M&amp;rft.aulast=Brudno&amp;rft.date=2003&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.pages=66&amp;rft.volume=4&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-4-66&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-edgar-18"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-edgar_18-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Edgar RC (2004). <a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC390337">"MUSCLE: multiple sequence alignment with high accuracy and high throughput"</a>. <i>Nucleic Acids Research</i>. <b>32</b> (5): 1792–97. <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC390337">390337</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/15034147">15034147</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkh340">10.1093/nar/gkh340</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=MUSCLE%3A+multiple+sequence+alignment+with+high+accuracy+and+high+throughput&amp;rft.au=Edgar+RC&amp;rft.date=2004&amp;rft.genre=article&amp;rft.issue=5&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.pages=1792-97&amp;rft.volume=32&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC390337&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC390337&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkh340&amp;rft_id=info%3Apmid%2F15034147&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-mergealign-19"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-mergealign_19-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Collingridge PW, Kelly S (2012). <a rel="nofollow" class="external text" href="http://www.biomedcentral.com/1471-2105/13/117">"MergeAlign: improving multiple sequence alignment performance by dynamic reconstruction of consensus multiple sequence alignments"</a>. <i>BMC Bioinformatics</i>. <b>13</b> (117). <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413523">3413523</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/22646090">22646090</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-13-117">10.1186/1471-2105-13-117</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=MergeAlign%3A+improving+multiple+sequence+alignment+performance+by+dynamic+reconstruction+of+consensus+multiple+sequence+alignments&amp;rft.au=Kelly%2C+S&amp;rft.aufirst=PW&amp;rft.aulast=Collingridge&amp;rft.date=2012&amp;rft.genre=article&amp;rft.issue=117&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.volume=13&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3413523&amp;rft_id=http%3A%2F%2Fwww.biomedcentral.com%2F1471-2105%2F13%2F117&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-13-117&amp;rft_id=info%3Apmid%2F22646090&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-hughey-20"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-hughey_20-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Hughey R, Krogh A (1996). "Hidden Markov models for sequence analysis: extension and analysis of the basic method". <i>CABIOS</i>. <b>12</b> (2): 95–107. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/8744772">8744772</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2F12.2.95">10.1093/bioinformatics/12.2.95</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Hidden+Markov+models+for+sequence+analysis%3A+extension+and+analysis+of+the+basic+method&amp;rft.au=Krogh%2C+A&amp;rft.aufirst=R&amp;rft.aulast=Hughey&amp;rft.date=1996&amp;rft.genre=article&amp;rft.issue=2&amp;rft.jtitle=CABIOS&amp;rft.pages=95-107&amp;rft.volume=12&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2F12.2.95&amp;rft_id=info%3Apmid%2F8744772&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-grasso-21"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-grasso_21-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Grasso C, Lee C (2004). "Combining partial order alignment and progressive multiple sequence alignment increases alignment speed and scalability to very large alignment problems". <i>Bioinformatics</i>. <b>20</b> (10): 1546–56. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/14962922">14962922</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbth126">10.1093/bioinformatics/bth126</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Combining+partial+order+alignment+and+progressive+multiple+sequence+alignment+increases+alignment+speed+and+scalability+to+very+large+alignment+problems&amp;rft.au=Lee%2C+C&amp;rft.aufirst=C&amp;rft.aulast=Grasso&amp;rft.date=2004&amp;rft.genre=article&amp;rft.issue=10&amp;rft.jtitle=Bioinformatics&amp;rft.pages=1546-56&amp;rft.volume=20&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbth126&amp;rft_id=info%3Apmid%2F14962922&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-hugheyT-22"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-hugheyT_22-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text">Hughey R, Krogh A. SAM: Sequence alignment and modeling software system. Technical Report UCSC-CRL-96-22, University of California, Santa Cruz, CA, September 1996.</span></li>
<li id="cite_note-durbin-23"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-durbin_23-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text">Durbin R, Eddy S, Krogh A, Mitchison G. (1998). Biological sequence analysis: probabilistic models of proteins and nucleic acids, Cambridge University Press, 1998.</span></li>
<li id="cite_note-24"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-24"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Söding J (2005). "Protein homology detection by HMM-HMM comparison". <i>Bioinformatics</i>. <b>21</b> (7): 951–960. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/15531603">15531603</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbti125">10.1093/bioinformatics/bti125</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Protein+homology+detection+by+HMM-HMM+comparison&amp;rft.au=S%C3%B6ding+J&amp;rft.date=2005&amp;rft.genre=article&amp;rft.issue=7&amp;rft.jtitle=Bioinformatics&amp;rft.pages=951-960&amp;rft.volume=21&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbti125&amp;rft_id=info%3Apmid%2F15531603&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-25"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-25"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Battey JN, Kopp J, Bordoli L, Read RJ, Clarke ND, Schwede T (2007). "Automated server predictions in CASP7". <i>Proteins</i>. <b>69</b> (Suppl 8): 68–82. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/17894354">17894354</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1002%2Fprot.21761">10.1002/prot.21761</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Automated+server+predictions+in+CASP7&amp;rft.au=Bordoli%2C+L&amp;rft.au=Clarke%2C+ND&amp;rft.au=Kopp%2C+J&amp;rft.au=Read%2C+RJ&amp;rft.au=Schwede%2C+T&amp;rft.aufirst=JN&amp;rft.aulast=Battey&amp;rft.date=2007&amp;rft.genre=article&amp;rft.issue=Suppl+8&amp;rft.jtitle=Proteins&amp;rft.pages=68-82&amp;rft.volume=69&amp;rft_id=info%3Adoi%2F10.1002%2Fprot.21761&amp;rft_id=info%3Apmid%2F17894354&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-notredame2-26"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-notredame2_26-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Notredame C, Higgins DG (1996). <a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC145823">"SAGA: sequence alignment by genetic algorithm"</a>. <i>Nucleic Acids Res</i>. <b>24</b> (8): 1515–24. <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC145823">145823</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/8628686">8628686</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2F24.8.1515">10.1093/nar/24.8.1515</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=SAGA%3A+sequence+alignment+by+genetic+algorithm&amp;rft.au=Higgins%2C+DG&amp;rft.aufirst=C&amp;rft.aulast=Notredame&amp;rft.date=1996&amp;rft.genre=article&amp;rft.issue=8&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.pages=1515-24&amp;rft.volume=24&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC145823&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC145823&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F24.8.1515&amp;rft_id=info%3Apmid%2F8628686&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-notredame3-27"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-notredame3_27-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Notredame C, O'Brien EA, Higgins DG (1997). <a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC147093">"RAGA: RNA sequence alignment by genetic algorithm"</a>. <i>Nucleic Acids Res</i>. <b>25</b> (22): 4570–80. <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC147093">147093</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/9358168">9358168</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2F25.22.4570">10.1093/nar/25.22.4570</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=RAGA%3A+RNA+sequence+alignment+by+genetic+algorithm&amp;rft.au=Higgins%2C+DG&amp;rft.au=O%27Brien%2C+EA&amp;rft.aufirst=C&amp;rft.aulast=Notredame&amp;rft.date=1997&amp;rft.genre=article&amp;rft.issue=22&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.pages=4570-80&amp;rft.volume=25&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC147093&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC147093&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F25.22.4570&amp;rft_id=info%3Apmid%2F9358168&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-kim-28"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-kim_28-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Kim J, Pramanik S, Chung MJ (1994). "Multiple sequence alignment using simulated annealing". <i>Comput Appl Biosci</i>. <b>10</b> (4): 419–26. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/7804875">7804875</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2F10.4.419">10.1093/bioinformatics/10.4.419</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Multiple+sequence+alignment+using+simulated+annealing&amp;rft.au=Chung%2C+MJ&amp;rft.au=Pramanik%2C+S&amp;rft.aufirst=J&amp;rft.aulast=Kim&amp;rft.date=1994&amp;rft.genre=article&amp;rft.issue=4&amp;rft.jtitle=Comput+Appl+Biosci&amp;rft.pages=419-26&amp;rft.volume=10&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2F10.4.419&amp;rft_id=info%3Apmid%2F7804875&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-29"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-29"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><a rel="nofollow" class="external text" href="http://www.dwavesys.com/press-releases/d-wave-initiates-open-quantum-software-environment">D-Wave Initiates Open Quantum Software Environment 11 January 2017</a></span></li>
<li id="cite_note-Loytynoja-2005-30"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-Loytynoja-2005_30-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Loytynoja, A. (2005). "An algorithm for progressive multiple alignment of sequences with insertions". <i>Proceedings of the National Academy of Sciences</i>. <b>102</b> (30): 10557–10562. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1073%2Fpnas.0409137102">10.1073/pnas.0409137102</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=An+algorithm+for+progressive+multiple+alignment+of+sequences+with+insertions&amp;rft.aufirst=A.&amp;rft.aulast=Loytynoja&amp;rft.date=2005&amp;rft.genre=article&amp;rft.issue=30&amp;rft.jtitle=Proceedings+of+the+National+Academy+of+Sciences&amp;rft.pages=10557-10562&amp;rft.volume=102&amp;rft_id=info%3Adoi%2F10.1073%2Fpnas.0409137102&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-Loytynoja-2008-31"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-Loytynoja-2008_31-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Loytynoja, A.; Goldman, N. (2008). "Phylogeny-Aware Gap Placement Prevents Errors in Sequence Alignment and Evolutionary Analysis". <i>Science</i>. <b>320</b> (5883): 1632–1635. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/18566285">18566285</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1126%2Fscience.1158395">10.1126/science.1158395</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Phylogeny-Aware+Gap+Placement+Prevents+Errors+in+Sequence+Alignment+and+Evolutionary+Analysis&amp;rft.au=Goldman%2C+N.&amp;rft.aufirst=A.&amp;rft.aulast=Loytynoja&amp;rft.date=2008&amp;rft.genre=article&amp;rft.issue=5883&amp;rft.jtitle=Science&amp;rft.pages=1632-1635&amp;rft.volume=320&amp;rft_id=info%3Adoi%2F10.1126%2Fscience.1158395&amp;rft_id=info%3Apmid%2F18566285&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-Loytynoja-2012-32"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-Loytynoja-2012_32-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Loytynoja, A.; Vilella, A. J.; Goldman, N. (2012). <a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3381962">"Accurate extension of multiple sequence alignments using a phylogeny-aware graph algorithm"</a>. <i>Bioinformatics</i>. <b>28</b> (13): 1684–1691. <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3381962">3381962</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/22531217">22531217</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts198">10.1093/bioinformatics/bts198</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Accurate+extension+of+multiple+sequence+alignments+using+a+phylogeny-aware+graph+algorithm&amp;rft.au=Goldman%2C+N.&amp;rft.au=Vilella%2C+A.+J.&amp;rft.aufirst=A.&amp;rft.aulast=Loytynoja&amp;rft.date=2012&amp;rft.genre=article&amp;rft.issue=13&amp;rft.jtitle=Bioinformatics&amp;rft.pages=1684-1691&amp;rft.volume=28&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3381962&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3381962&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbts198&amp;rft_id=info%3Apmid%2F22531217&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-Szalkowski-2012-33"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-Szalkowski-2012_33-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Szalkowski, A. M. (2012). <a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3495709">"Fast and robust multiple sequence alignment with phylogenyaware gap placement"</a>. <i>BMC Bioinformatics</i>. <b>13</b>: 129–1180. <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3495709">3495709</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/22694311">22694311</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-13-129">10.1186/1471-2105-13-129</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Fast+and+robust+multiple+sequence+alignment+with+phylogenyaware+gap+placement&amp;rft.aufirst=A.+M.&amp;rft.aulast=Szalkowski&amp;rft.date=2012&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.pages=129-1180&amp;rft.volume=13&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3495709&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC3495709&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-13-129&amp;rft_id=info%3Apmid%2F22694311&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-henikoff1991-34"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-henikoff1991_34-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Henikoff S, Henikoff JG (1991). <a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC329220">"Automated assembly of protein blocks for database searching"</a>. <i>Nucleic Acids Res</i>. <b>19</b> (23): 6565–6572. <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC329220">329220</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/1754394">1754394</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2F19.23.6565">10.1093/nar/19.23.6565</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Automated+assembly+of+protein+blocks+for+database+searching&amp;rft.au=Henikoff%2C+JG&amp;rft.aufirst=S&amp;rft.aulast=Henikoff&amp;rft.date=1991&amp;rft.genre=article&amp;rft.issue=23&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.pages=6565-6572&amp;rft.volume=19&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC329220&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC329220&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F19.23.6565&amp;rft_id=info%3Apmid%2F1754394&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-baileyelkan1994-35"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-baileyelkan1994_35-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation book">Bailey TL, Elkan C (1994). "Fitting a mixture model by expectation maximization to discover motifs in biopolymers". <i>Proceedings of the Second International Conference on Intelligent Systems for Molecular Biology</i>. Menlo Park, California: AAAI Press. pp.&nbsp;28–36.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Fitting+a+mixture+model+by+expectation+maximization+to+discover+motifs+in+biopolymers&amp;rft.au=Elkan%2C+C&amp;rft.aufirst=TL&amp;rft.aulast=Bailey&amp;rft.btitle=Proceedings+of+the+Second+International+Conference+on+Intelligent+Systems+for+Molecular+Biology&amp;rft.date=1994&amp;rft.genre=bookitem&amp;rft.pages=28-36&amp;rft.place=Menlo+Park%2C+California&amp;rft.pub=AAAI+Press&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-baileygribskov1998-36"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-baileygribskov1998_36-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Bailey TL, Gribskov M (1998). "Combining evidence using p-values: application to sequence homology searches". <i>Bioinformatics</i>. <b>14</b> (1): 48–54. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/9520501">9520501</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2F14.1.48">10.1093/bioinformatics/14.1.48</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Combining+evidence+using+p-values%3A+application+to+sequence+homology+searches&amp;rft.au=Gribskov%2C+M&amp;rft.aufirst=TL&amp;rft.aulast=Bailey&amp;rft.date=1998&amp;rft.genre=article&amp;rft.issue=1&amp;rft.jtitle=Bioinformatics&amp;rft.pages=48-54&amp;rft.volume=14&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2F14.1.48&amp;rft_id=info%3Apmid%2F9520501&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-Salama2013-37"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-Salama2013_37-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite id="CITEREFSalamaStekel2013" class="citation">Salama RA, Stekel DJ (2013), "A non-independent energy-based multiple sequence alignment improves prediction of transcription factor binding sites", <i>Bioinformatics</i>, <b>29</b>: 2699–2704, <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/23990411">23990411</a>, <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtt463">10.1093/bioinformatics/btt463</a></cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=A+non-independent+energy-based+multiple+sequence+alignment+improves+prediction+of+transcription+factor+binding+sites&amp;rft.au=Stekel%2C+DJ&amp;rft.aufirst=RA&amp;rft.aulast=Salama&amp;rft.date=2013&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.pages=2699-2704&amp;rft.volume=29&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2Fbtt463&amp;rft_id=info%3Apmid%2F23990411&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-nuin2006-38"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-nuin2006_38-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Nuin PA, Wang Z, Tillier ER (2006). <a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1633746">"The accuracy of several multiple sequence alignment programs for proteins"</a>. <i>BMC Bioinformatics</i>. <b>7</b>: 471. <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1633746">1633746</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/17062146">17062146</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1186%2F1471-2105-7-471">10.1186/1471-2105-7-471</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=The+accuracy+of+several+multiple+sequence+alignment+programs+for+proteins&amp;rft.au=Tillier%2C+ER&amp;rft.au=Wang%2C+Z&amp;rft.aufirst=PA&amp;rft.aulast=Nuin&amp;rft.date=2006&amp;rft.genre=article&amp;rft.jtitle=BMC+Bioinformatics&amp;rft.pages=471&amp;rft.volume=7&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1633746&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1633746&amp;rft_id=info%3Adoi%2F10.1186%2F1471-2105-7-471&amp;rft_id=info%3Apmid%2F17062146&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-39"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-39"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation web"><a rel="nofollow" class="external text" href="http://www.embl.de/~seqanal/MSAcambridgeGenetics2007/MSAmanualAdjustments/MSAmanualAdjustments.html">"Manual editing and adjustment of MSAs"</a>. European Molecular Biology Laboratory. 2007<span class="reference-accessdate">. Retrieved <span class="nowrap">March 7,</span> 2010</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.btitle=Manual+editing+and+adjustment+of+MSAs&amp;rft.date=2007&amp;rft.genre=unknown&amp;rft.pub=European+Molecular+Biology+Laboratory&amp;rft_id=http%3A%2F%2Fwww.embl.de%2F~seqanal%2FMSAcambridgeGenetics2007%2FMSAmanualAdjustments%2FMSAmanualAdjustments.html&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-castresana2000-40"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-castresana2000_40-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Castresana J (2000). "Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis". <i>Mol Biol Evol</i>. <b>17</b>: 540–552. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Foxfordjournals.molbev.a026334">10.1093/oxfordjournals.molbev.a026334</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Selection+of+conserved+blocks+from+multiple+alignments+for+their+use+in+phylogenetic+analysis&amp;rft.au=Castresana+J&amp;rft.date=2000&amp;rft.genre=article&amp;rft.jtitle=Mol+Biol+Evol&amp;rft.pages=540-552&amp;rft.volume=17&amp;rft_id=info%3Adoi%2F10.1093%2Foxfordjournals.molbev.a026334&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-loytynojaMilinkovitch2001-41"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-loytynojaMilinkovitch2001_41-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite id="CITEREFLoytynojaMilinkovitch2001" class="citation">Loytynoja A, Milinkovitch MC (2001), "SOAP, cleaning multiple alignments from unstable blocks", <i>Bioinformatics</i>, <b>17</b>: 573–574, <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fbioinformatics%2F17.6.573">10.1093/bioinformatics/17.6.573</a></cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=SOAP%2C+cleaning+multiple+alignments+from+unstable+blocks&amp;rft.au=Milinkovitch%2C+MC&amp;rft.aufirst=A&amp;rft.aulast=Loytynoja&amp;rft.date=2001&amp;rft.genre=article&amp;rft.jtitle=Bioinformatics&amp;rft.pages=573-574&amp;rft.volume=17&amp;rft_id=info%3Adoi%2F10.1093%2Fbioinformatics%2F17.6.573&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-poirotOTooleNotredame2003-42"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-poirotOTooleNotredame2003_42-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite id="CITEREFPoirot_OO.E2.80.99Toole_ENotredame_C2003" class="citation">Poirot O; O’Toole E; Notredame C (2003), "Tcoffee@igs: a web server for computing, evaluating and combining multiple sequence alignments", <i>Nucleic Acids Res</i>, <b>31</b>: 3503–3506, <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkg522">10.1093/nar/gkg522</a></cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Tcoffee%40igs%3A+a+web+server+for+computing%2C+evaluating+and+combining+multiple+sequence+alignments&amp;rft.au=Notredame+C&amp;rft.au=O%E2%80%99Toole+E&amp;rft.au=Poirot+O&amp;rft.date=2003&amp;rft.genre=article&amp;rft.jtitle=Nucleic+Acids+Res&amp;rft.pages=3503-3506&amp;rft.volume=31&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkg522&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-TCS2014MBE-43"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-TCS2014MBE_43-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Chang, JM; Di Tommaso, P; Notredame, C (Jun 2014). <a rel="nofollow" class="external text" href="http://mbe.oxfordjournals.org/content/31/6/1625">"TCS: A New Multiple Sequence Alignment Reliability Measure to Estimate Alignment Accuracy and Improve Phylogenetic Tree Reconstruction."</a>. <i>Molecular Biology and Evolution</i>. <b>31</b> (6): 1625–37. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/24694831">24694831</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fmolbev%2Fmsu117">10.1093/molbev/msu117</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=TCS%3A+A+New+Multiple+Sequence+Alignment+Reliability+Measure+to+Estimate+Alignment+Accuracy+and+Improve+Phylogenetic+Tree+Reconstruction.&amp;rft.au=Di+Tommaso%2C+P&amp;rft.au=Notredame%2C+C&amp;rft.aufirst=JM&amp;rft.aulast=Chang&amp;rft.date=2014-06&amp;rft.genre=article&amp;rft.issue=6&amp;rft.jtitle=Molecular+Biology+and+Evolution&amp;rft.pages=1625-37&amp;rft.volume=31&amp;rft_id=http%3A%2F%2Fmbe.oxfordjournals.org%2Fcontent%2F31%2F6%2F1625&amp;rft_id=info%3Adoi%2F10.1093%2Fmolbev%2Fmsu117&amp;rft_id=info%3Apmid%2F24694831&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-TCS_2015_NAR-44"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-TCS_2015_NAR_44-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Chang, JM; Di Tommaso, P; Lefort, V; Gascuel, O; Notredame, C (1 July 2015). <a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4489230">"TCS: a web server for multiple sequence alignment evaluation and phylogenetic reconstruction."</a>. <i>Nucleic Acids Research</i>. <b>43</b> (W1): W3–6. <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4489230">4489230</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/25855806">25855806</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2Fgkv310">10.1093/nar/gkv310</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=TCS%3A+a+web+server+for+multiple+sequence+alignment+evaluation+and+phylogenetic+reconstruction.&amp;rft.au=Di+Tommaso%2C+P&amp;rft.au=Gascuel%2C+O&amp;rft.au=Lefort%2C+V&amp;rft.au=Notredame%2C+C&amp;rft.aufirst=JM&amp;rft.aulast=Chang&amp;rft.date=2015-07-01&amp;rft.genre=article&amp;rft.issue=W1&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.pages=W3-6&amp;rft.volume=43&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4489230&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC4489230&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2Fgkv310&amp;rft_id=info%3Apmid%2F25855806&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-bradley2009-45"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-bradley2009_45-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite id="CITEREFBradley_RK.2C_Roberts_A.2C_Smoot_M.2C_Juvekar_S.2C_Do_J.2C_Dewey_C.2C_Holmes_I.2C_Pachter_L2009" class="citation">Bradley RK, Roberts A, Smoot M, Juvekar S, Do J, Dewey C, Holmes I, <a href="https://en.wikipedia.org/wiki/Lior_Pachter" title="Lior Pachter">Pachter L</a> (2009), "Fast statistical alignment", <i>PLoS Comput Biol</i>, <b>5</b>: e1000392, <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2684580">2684580</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>, <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/19478997">19478997</a>, <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1371%2Fjournal.pcbi.1000392">10.1371/journal.pcbi.1000392</a></cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Fast+statistical+alignment&amp;rft.au=Bradley+RK%2C+Roberts+A%2C+Smoot+M%2C+Juvekar+S%2C+Do+J%2C+Dewey+C%2C+Holmes+I%2C+Pachter+L&amp;rft.date=2009&amp;rft.genre=article&amp;rft.jtitle=PLoS+Comput+Biol&amp;rft.pages=e1000392&amp;rft.volume=5&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC2684580&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pcbi.1000392&amp;rft_id=info%3Apmid%2F19478997&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span><span class="citation-comment" style="display:none; color:#33aa33; margin-left:0.3em">CS1 maint: Multiple names: authors list (<a href="https://en.wikipedia.org/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">link</a>)</span></span></li>
<li id="cite_note-landanGraur2008-46"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-landanGraur2008_46-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite id="CITEREFLandanGraur2008" class="citation">Landan G, Graur D (2008), "Local reliability measures from sets of co-optimal multiple sequence alignments", <i>Pac Symp Biocomput</i>, <b>13</b>: 15–24, <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1142%2F9789812776136_0003">10.1142/9789812776136_0003</a></cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Local+reliability+measures+from+sets+of+co-optimal+multiple+sequence+alignments&amp;rft.au=Graur%2C+D&amp;rft.aufirst=G&amp;rft.aulast=Landan&amp;rft.date=2008&amp;rft.genre=article&amp;rft.jtitle=Pac+Symp+Biocomput&amp;rft.pages=15-24&amp;rft.volume=13&amp;rft_id=info%3Adoi%2F10.1142%2F9789812776136_0003&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-penn2010-47"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-penn2010_47-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite class="citation journal">Penn O, Privman E, Landan G, Graur D, Pupko T (2010). "An Alignment Confidence Score Capturing Robustness to Guide Tree Uncertainty". <i>Mol Biol Evol</i>. <b>27</b>: 1759–1767. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fmolbev%2Fmsq066">10.1093/molbev/msq066</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=An+Alignment+Confidence+Score+Capturing+Robustness+to+Guide+Tree+Uncertainty&amp;rft.au=Graur%2C+D&amp;rft.au=Landan%2C+G&amp;rft.au=Privman%2C+E&amp;rft.au=Pupko%2C+T&amp;rft.aufirst=O&amp;rft.aulast=Penn&amp;rft.date=2010&amp;rft.genre=article&amp;rft.jtitle=Mol+Biol+Evol&amp;rft.pages=1759-1767&amp;rft.volume=27&amp;rft_id=info%3Adoi%2F10.1093%2Fmolbev%2Fmsq066&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-redelingsSuchard2005-48"><span class="mw-cite-backlink"><b><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-redelingsSuchard2005_48-0"><span class="cite-accessibility-label">Jump up </span>^</a></b></span> <span class="reference-text"><cite id="CITEREFRedelingsSuchard2005" class="citation">Redelings BD, Suchard MA (2005), "Joint Bayesian Estimation of Alignment and Phylogeny", <i>Systematic Biology</i>, <b>54</b>: 401–418, <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1080%2F10635150590947041">10.1080/10635150590947041</a></cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Joint+Bayesian+Estimation+of+Alignment+and+Phylogeny&amp;rft.au=Suchard%2C+MA&amp;rft.aufirst=BD&amp;rft.aulast=Redelings&amp;rft.date=2005&amp;rft.genre=article&amp;rft.jtitle=Systematic+Biology&amp;rft.pages=401-418&amp;rft.volume=54&amp;rft_id=info%3Adoi%2F10.1080%2F10635150590947041&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
<li id="cite_note-Budd_2009-49"><span class="mw-cite-backlink">^ <a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-Budd_2009_49-0"><span class="cite-accessibility-label">Jump up to: </span><sup><i><b>a</b></i></sup></a> <a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment#cite_ref-Budd_2009_49-1"><sup><i><b>b</b></i></sup></a></span> <span class="reference-text"><cite class="citation web">Budd, Aidan (10 February 2009). <a rel="nofollow" class="external text" href="http://www.embl.de/~seqanal/courses/commonCourseContent/commonMsaExercises.html">"Multiple sequence alignment exercises and demonstrations"</a>. European Molecular Biology Laboratory<span class="reference-accessdate">. Retrieved <span class="nowrap">June 30,</span> 2010</span>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.aufirst=Aidan&amp;rft.aulast=Budd&amp;rft.btitle=Multiple+sequence+alignment+exercises+and+demonstrations&amp;rft.date=2009-02-10&amp;rft.genre=unknown&amp;rft.pub=European+Molecular+Biology+Laboratory&amp;rft_id=http%3A%2F%2Fwww.embl.de%2F~seqanal%2Fcourses%2FcommonCourseContent%2FcommonMsaExercises.html&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&nbsp;</span></span></span></li>
</ol>
</div>
<h3><span class="mw-headline" id="Survey_articles">Survey articles</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=15" title="Edit section: Survey articles">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<ul>
<li><cite class="citation book">Duret, L.; S. Abdeddaim (2000). "Multiple alignment for structural functional or phylogenetic analyses of homologous sequences". In D. Higgins and W. Taylor. <i>Bioinformatics sequence structure and databanks</i>. Oxford: Oxford University Press.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Multiple+alignment+for+structural+functional+or+phylogenetic+analyses+of+homologous+sequences&amp;rft.au=S.+Abdeddaim&amp;rft.aufirst=L.&amp;rft.aulast=Duret&amp;rft.btitle=Bioinformatics+sequence+structure+and+databanks&amp;rft.date=2000&amp;rft.genre=bookitem&amp;rft.place=Oxford&amp;rft.pub=Oxford+University+Press&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Abook" class="Z3988"><span style="display:none;">&nbsp;</span></span></li>
<li><cite class="citation journal">Notredame, C. (2002). "Recent progresses in multiple sequence alignment: a survey". <i>Pharmacogenomics</i>. <b>3</b> (1): 131–144. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/11966409">11966409</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1517%2F14622416.3.1.131">10.1517/14622416.3.1.131</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Recent+progresses+in+multiple+sequence+alignment%3A+a+survey&amp;rft.aufirst=C.&amp;rft.aulast=Notredame&amp;rft.date=2002&amp;rft.genre=article&amp;rft.issue=1&amp;rft.jtitle=Pharmacogenomics&amp;rft.pages=131-144&amp;rft.volume=3&amp;rft_id=info%3Adoi%2F10.1517%2F14622416.3.1.131&amp;rft_id=info%3Apmid%2F11966409&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></li>
<li><cite class="citation journal">Thompson, J. D.; Plewniak, F.; Poch, O. (1999). <a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC148477">"A comprehensive comparison of multiple sequence alignment programs"</a>. <i>Nucleic Acids Research</i>. <b>27</b> (13): 12682–2690. <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC148477">148477</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/10373585">10373585</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1093%2Fnar%2F27.13.2682">10.1093/nar/27.13.2682</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=A+comprehensive+comparison+of+multiple+sequence+alignment+programs&amp;rft.au=Plewniak%2C+F.&amp;rft.au=Poch%2C+O.&amp;rft.aufirst=J.+D.&amp;rft.aulast=Thompson&amp;rft.date=1999&amp;rft.genre=article&amp;rft.issue=13&amp;rft.jtitle=Nucleic+Acids+Research&amp;rft.pages=12682-2690&amp;rft.volume=27&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC148477&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC148477&amp;rft_id=info%3Adoi%2F10.1093%2Fnar%2F27.13.2682&amp;rft_id=info%3Apmid%2F10373585&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></li>
<li><cite class="citation journal">Wallace, I.M.; Blackshields, G.; Higgins, D.G. (2005). "Multiple sequence alignments". <i>Curr Opin Struct Biol</i>. <b>15</b> (3): 261–266. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/15963889">15963889</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1016%2Fj.sbi.2005.04.002">10.1016/j.sbi.2005.04.002</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Multiple+sequence+alignments&amp;rft.au=Blackshields%2C+G.&amp;rft.au=Higgins%2C+D.G.&amp;rft.aufirst=I.M.&amp;rft.aulast=Wallace&amp;rft.date=2005&amp;rft.genre=article&amp;rft.issue=3&amp;rft.jtitle=Curr+Opin+Struct+Biol&amp;rft.pages=261-266&amp;rft.volume=15&amp;rft_id=info%3Adoi%2F10.1016%2Fj.sbi.2005.04.002&amp;rft_id=info%3Apmid%2F15963889&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></li>
<li><cite class="citation journal">Notredame, C (2007). <a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1963500">"Recent Evolutions of Multiple Sequence Alignment Algorithms"</a>. <i>PLOS Computational Biology</i>. <b>3</b>: e123. <a href="https://en.wikipedia.org/wiki/PubMed_Central" title="PubMed Central">PMC</a>&nbsp;<span class="plainlinks"><a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1963500">1963500</a> <img alt="Freely accessible" src="./Multiple sequence alignment - Wikipedia_files/9px-Lock-green.svg.png" title="Freely accessible" width="9" height="14" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/14px-Lock-green.svg.png 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/6/65/Lock-green.svg/18px-Lock-green.svg.png 2x" data-file-width="512" data-file-height="813"></span>. <a href="https://en.wikipedia.org/wiki/PubMed_Identifier" class="mw-redirect" title="PubMed Identifier">PMID</a>&nbsp;<a rel="nofollow" class="external text" href="https://www.ncbi.nlm.nih.gov/pubmed/17784778">17784778</a>. <a href="https://en.wikipedia.org/wiki/Digital_object_identifier" title="Digital object identifier">doi</a>:<a rel="nofollow" class="external text" href="https://doi.org/10.1371%2Fjournal.pcbi.0030123">10.1371/journal.pcbi.0030123</a>.</cite><span title="ctx_ver=Z39.88-2004&amp;rfr_id=info%3Asid%2Fen.wikipedia.org%3AMultiple+sequence+alignment&amp;rft.atitle=Recent+Evolutions+of+Multiple+Sequence+Alignment+Algorithms&amp;rft.aufirst=C&amp;rft.aulast=Notredame&amp;rft.date=2007&amp;rft.genre=article&amp;rft.jtitle=PLOS+Computational+Biology&amp;rft.pages=e123&amp;rft.volume=3&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1963500&amp;rft_id=%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC1963500&amp;rft_id=info%3Adoi%2F10.1371%2Fjournal.pcbi.0030123&amp;rft_id=info%3Apmid%2F17784778&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal" class="Z3988"><span style="display:none;">&nbsp;</span></span></li>
</ul>
<h2><span class="mw-headline" id="External_links">External links</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=16" title="Edit section: External links">edit</a><span class="mw-editsection-bracket">]</span></span></h2>
<ul>
<li><a rel="nofollow" class="external text" href="http://www.expasy.org/tools/#align">ExPASy sequence alignment tools</a></li>
<li><a rel="nofollow" class="external text" href="https://web.archive.org/web/20100107231637/http://www.techfak.uni-bielefeld.de/bcd/Curric/MulAli/welcome.html">Archived Multiple Alignment Resource Page</a>&nbsp;— from the Virtual School of Natural Sciences</li>
<li><a rel="nofollow" class="external text" href="http://pbil.univ-lyon1.fr/alignment.html">Tools for Multiple Alignments</a>&nbsp;— from Pôle Bioinformatique Lyonnais</li>
<li><a rel="nofollow" class="external text" href="http://www.clustal.org/">An entry point to clustal servers and information</a></li>
<li><a rel="nofollow" class="external text" href="http://www.tcoffee.org/">An entry point to the main T-Coffee servers</a></li>
<li><a rel="nofollow" class="external text" href="http://www.mergealign.com/">An entry point to the main MergeAlign server and information</a></li>
<li>European Bioinformatics Institute servers:
<ul>
<li><a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/Tools/clustalw2/">ClustalW2</a>&nbsp;— general purpose multiple sequence alignment program for DNA or proteins.</li>
<li><a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/Tools/muscle/">Muscle</a>&nbsp;— MUltiple Sequence Comparison by Log-Expectation</li>
<li><a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/Tools/t-coffee/">T-coffee</a>&nbsp;— multiple sequence alignment.</li>
<li><a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/Tools/mafft/">MAFFT</a>&nbsp;— Multiple Alignment using Fast Fourier Transform</li>
<li><a rel="nofollow" class="external text" href="http://www.ebi.ac.uk/Tools/kalign/">KALIGN</a>&nbsp;— a fast and accurate multiple sequence alignment algorithm.</li>
</ul>
</li>
</ul>
<h3><span class="mw-headline" id="Lecture_notes.2C_tutorials.2C_and_courses">Lecture notes, tutorials, and courses</span><span class="mw-editsection"><span class="mw-editsection-bracket">[</span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit&amp;section=17" title="Edit section: Lecture notes, tutorials, and courses">edit</a><span class="mw-editsection-bracket">]</span></span></h3>
<ul>
<li><a rel="nofollow" class="external text" href="http://cmb.molgen.mpg.de/">Multiple sequence alignment lectures</a>&nbsp;— from the Max Planck Institute for Molecular Genetics</li>
<li><a rel="nofollow" class="external text" href="http://www.embl.de/~seqanal/courses/commonCourseContent/commonMsaExercises.html">Lecture Notes and practical exercises</a> on multiple sequence alignments at the <a href="https://en.wikipedia.org/wiki/European_Molecular_Biology_Laboratory" title="European Molecular Biology Laboratory">EMBL</a></li>
<li><a rel="nofollow" class="external text" href="http://www.avatar.se/molbioinfo2001/index.html">Molecular Bioinformatics Lecture Notes</a></li>
<li><a rel="nofollow" class="external text" href="http://bioinf.may.ie/school02/notes.html">Molecular Evolution and Bioinformatics Lecture Notes</a></li>
</ul>


<!-- 
NewPP limit report
Parsed by mw1184
Cached time: 20170714221232
Cache expiry: 2592000
Dynamic content: false
CPU time usage: 0.520 seconds
Real time usage: 0.570 seconds
Preprocessor visited node count: 2289/1000000
Preprocessor generated node count: 0/1500000
Post‐expand include size: 107105/2097152 bytes
Template argument size: 186/2097152 bytes
Highest expansion depth: 6/40
Expensive parser function count: 0/500
Lua time usage: 0.297/10.000 seconds
Lua memory usage: 3.03 MB/50 MB
-->
<!--
Transclusion expansion time report (%,ms,calls,template)
100.00%  460.013      1 -total
 84.14%  387.036      1 Template:Reflist
 47.82%  219.962     37 Template:Cite_journal
 14.74%   67.815      5 Template:Cite_web
  9.02%   41.493      1 Template:Good_article
  8.71%   40.087      1 Template:Top_icon
  8.31%   38.213      6 Template:Citation
  6.43%   29.576      1 Template:Category_handler
  3.04%   13.964      2 Template:Cite_book
  0.46%    2.110      1 Template:Column-width
-->
</div>
<!-- Saved in parser cache with key enwiki:pcache:idhash:4066308-0!canonical and timestamp 20170714221231 and revision id 774102939
 -->
<noscript>&lt;img src="//en.wikipedia.org/wiki/Special:CentralAutoLogin/start?type=1x1" alt="" title="" width="1" height="1" style="border: none; position: absolute;" /&gt;</noscript></div>					<div class="printfooter">
						Retrieved from "<a dir="ltr" href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;oldid=774102939">https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;oldid=774102939</a>"					</div>
				<div id="catlinks" class="catlinks" data-mw="interface"><div id="mw-normal-catlinks" class="mw-normal-catlinks"><a href="https://en.wikipedia.org/wiki/Help:Category" title="Help:Category">Categories</a>: <ul><li><a href="https://en.wikipedia.org/wiki/Category:Bioinformatics" title="Category:Bioinformatics">Bioinformatics</a></li><li><a href="https://en.wikipedia.org/wiki/Category:Computational_phylogenetics" title="Category:Computational phylogenetics">Computational phylogenetics</a></li><li><a href="https://en.wikipedia.org/wiki/Category:Markov_models" title="Category:Markov models">Markov models</a></li></ul></div><div id="mw-hidden-catlinks" class="mw-hidden-catlinks mw-hidden-cats-hidden">Hidden categories: <ul><li><a href="https://en.wikipedia.org/wiki/Category:CS1_maint:_Multiple_names:_authors_list" title="Category:CS1 maint: Multiple names: authors list">CS1 maint: Multiple names: authors list</a></li><li><a href="https://en.wikipedia.org/wiki/Category:Good_articles" title="Category:Good articles">Good articles</a></li></ul></div></div>				<div class="visualClear"></div>
							</div>
		</div>
		<div id="mw-navigation">
			<h2>Navigation menu</h2>

			<div id="mw-head">
									<div id="p-personal" role="navigation" class="" aria-labelledby="p-personal-label">
						<h3 id="p-personal-label">Personal tools</h3>
						<ul>
							<li id="pt-anonuserpage">Not logged in</li><li id="pt-anontalk"><a href="https://en.wikipedia.org/wiki/Special:MyTalk" title="Discussion about edits from this IP address [alt-shift-n]" accesskey="n">Talk</a></li><li id="pt-anoncontribs"><a href="https://en.wikipedia.org/wiki/Special:MyContributions" title="A list of edits made from this IP address [alt-shift-y]" accesskey="y">Contributions</a></li><li id="pt-createaccount"><a href="https://en.wikipedia.org/w/index.php?title=Special:CreateAccount&amp;returnto=Multiple+sequence+alignment" title="You are encouraged to create an account and log in; however, it is not mandatory">Create account</a></li><li id="pt-login"><a href="https://en.wikipedia.org/w/index.php?title=Special:UserLogin&amp;returnto=Multiple+sequence+alignment" title="You&#39;re encouraged to log in; however, it&#39;s not mandatory. [alt-shift-o]" accesskey="o">Log in</a></li>						</ul>
					</div>
									<div id="left-navigation">
										<div id="p-namespaces" role="navigation" class="vectorTabs" aria-labelledby="p-namespaces-label">
						<h3 id="p-namespaces-label">Namespaces</h3>
						<ul>
														<li id="ca-nstab-main" class="selected"><span><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment" title="View the content page [alt-shift-c]" accesskey="c">Article</a></span></li>
							<li id="ca-talk"><span><a href="https://en.wikipedia.org/wiki/Talk:Multiple_sequence_alignment" rel="discussion" title="Discussion about the content page [alt-shift-t]" accesskey="t">Talk</a></span></li>
						</ul>
					</div>
										<div id="p-variants" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-variants-label">
												<h3 id="p-variants-label" tabindex="0">
							<span>Variants</span>
						</h3>

						<div class="menu">
							<ul>
															</ul>
						</div>
					</div>
									</div>
				<div id="right-navigation">
										<div id="p-views" role="navigation" class="vectorTabs" aria-labelledby="p-views-label">
						<h3 id="p-views-label">Views</h3>
						<ul>
														<li id="ca-view" class="selected"><span><a href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment">Read</a></span></li>
							<li id="ca-edit"><span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=edit" title="Edit this page [alt-shift-e]" accesskey="e">Edit</a></span></li>
							<li id="ca-history" class="collapsible"><span><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=history" title="Past revisions of this page [alt-shift-h]" accesskey="h">View history</a></span></li>
						</ul>
					</div>
										<div id="p-cactions" role="navigation" class="vectorMenu emptyPortlet" aria-labelledby="p-cactions-label" style="">
						<h3 id="p-cactions-label" tabindex="0"><span>More</span></h3>

						<div class="menu">
							<ul>
															</ul>
						</div>
					</div>
										<div id="p-search" role="search">
						<h3>
							<label for="searchInput">Search</label>
						</h3>

						<form action="https://en.wikipedia.org/w/index.php" id="searchform">
							<div id="simpleSearch">
							<input type="search" name="search" placeholder="Search Wikipedia" title="Search Wikipedia [alt-shift-f]" accesskey="f" id="searchInput" tabindex="1" autocomplete="off"><input type="hidden" value="Special:Search" name="title"><input type="submit" name="go" value="Go" title="Go to a page with this exact name if it exists" id="searchButton" class="searchButton">							</div>
						</form>
					</div>
									</div>
			</div>
			<div id="mw-panel">
				<div id="p-logo" role="banner"><a class="mw-wiki-logo" href="https://en.wikipedia.org/wiki/Main_Page" title="Visit the main page"></a></div>
						<div class="portal" role="navigation" id="p-navigation" aria-labelledby="p-navigation-label">
			<h3 id="p-navigation-label">Navigation</h3>

			<div class="body">
									<ul>
						<li id="n-mainpage-description"><a href="https://en.wikipedia.org/wiki/Main_Page" title="Visit the main page [alt-shift-z]" accesskey="z">Main page</a></li><li id="n-contents"><a href="https://en.wikipedia.org/wiki/Portal:Contents" title="Guides to browsing Wikipedia">Contents</a></li><li id="n-featuredcontent"><a href="https://en.wikipedia.org/wiki/Portal:Featured_content" title="Featured content – the best of Wikipedia">Featured content</a></li><li id="n-currentevents"><a href="https://en.wikipedia.org/wiki/Portal:Current_events" title="Find background information on current events">Current events</a></li><li id="n-randompage"><a href="https://en.wikipedia.org/wiki/Special:Random" title="Load a random article [alt-shift-x]" accesskey="x">Random article</a></li><li id="n-sitesupport"><a href="https://donate.wikimedia.org/wiki/Special:FundraiserRedirector?utm_source=donate&amp;utm_medium=sidebar&amp;utm_campaign=C13_en.wikipedia.org&amp;uselang=en" title="Support us">Donate to Wikipedia</a></li><li id="n-shoplink"><a href="https://shop.wikimedia.org/" title="Visit the Wikipedia store">Wikipedia store</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id="p-interaction" aria-labelledby="p-interaction-label">
			<h3 id="p-interaction-label">Interaction</h3>

			<div class="body">
									<ul>
						<li id="n-help"><a href="https://en.wikipedia.org/wiki/Help:Contents" title="Guidance on how to use and edit Wikipedia">Help</a></li><li id="n-aboutsite"><a href="https://en.wikipedia.org/wiki/Wikipedia:About" title="Find out about Wikipedia">About Wikipedia</a></li><li id="n-portal"><a href="https://en.wikipedia.org/wiki/Wikipedia:Community_portal" title="About the project, what you can do, where to find things">Community portal</a></li><li id="n-recentchanges"><a href="https://en.wikipedia.org/wiki/Special:RecentChanges" title="A list of recent changes in the wiki [alt-shift-r]" accesskey="r">Recent changes</a></li><li id="n-contactpage"><a href="https://en.wikipedia.org/wiki/Wikipedia:Contact_us" title="How to contact Wikipedia">Contact page</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id="p-tb" aria-labelledby="p-tb-label">
			<h3 id="p-tb-label">Tools</h3>

			<div class="body">
									<ul>
						<li id="t-whatlinkshere"><a href="https://en.wikipedia.org/wiki/Special:WhatLinksHere/Multiple_sequence_alignment" title="List of all English Wikipedia pages containing links to this page [alt-shift-j]" accesskey="j">What links here</a></li><li id="t-recentchangeslinked"><a href="https://en.wikipedia.org/wiki/Special:RecentChangesLinked/Multiple_sequence_alignment" rel="nofollow" title="Recent changes in pages linked from this page [alt-shift-k]" accesskey="k">Related changes</a></li><li id="t-upload"><a href="https://en.wikipedia.org/wiki/Wikipedia:File_Upload_Wizard" title="Upload files [alt-shift-u]" accesskey="u">Upload file</a></li><li id="t-specialpages"><a href="https://en.wikipedia.org/wiki/Special:SpecialPages" title="A list of all special pages [alt-shift-q]" accesskey="q">Special pages</a></li><li id="t-permalink"><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;oldid=774102939" title="Permanent link to this revision of the page">Permanent link</a></li><li id="t-info"><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;action=info" title="More information about this page">Page information</a></li><li id="t-wikibase"><a href="https://www.wikidata.org/wiki/Special:EntityPage/Q1377767" title="Link to connected data repository item [alt-shift-g]" accesskey="g">Wikidata item</a></li><li id="t-cite"><a href="https://en.wikipedia.org/w/index.php?title=Special:CiteThisPage&amp;page=Multiple_sequence_alignment&amp;id=774102939" title="Information on how to cite this page">Cite this page</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id="p-coll-print_export" aria-labelledby="p-coll-print_export-label">
			<h3 id="p-coll-print_export-label">Print/export</h3>

			<div class="body">
									<ul>
						<li id="coll-create_a_book"><a href="https://en.wikipedia.org/w/index.php?title=Special:Book&amp;bookcmd=book_creator&amp;referer=Multiple+sequence+alignment">Create a book</a></li><li id="coll-download-as-rdf2latex"><a href="https://en.wikipedia.org/w/index.php?title=Special:ElectronPdf&amp;page=Multiple+sequence+alignment&amp;action=show-selection-screen&amp;coll-download-url=%2Fw%2Findex.php%3Ftitle%3DSpecial%3ABook%26bookcmd%3Drender_article%26arttitle%3DMultiple%2Bsequence%2Balignment%26returnto%3DMultiple%2Bsequence%2Balignment%26oldid%3D774102939%26writer%3Drdf2latex">Download as PDF</a></li><li id="t-print"><a href="https://en.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;printable=yes" title="Printable version of this page [alt-shift-p]" accesskey="p">Printable version</a></li>					</ul>
							</div>
		</div>
			<div class="portal" role="navigation" id="p-lang" aria-labelledby="p-lang-label"><span class="uls-settings-trigger" title="Language settings" tabindex="0" role="button" aria-haspopup="true"></span>
			<h3 id="p-lang-label">Languages</h3>

			<div class="body">
									<ul>
						<li class="interlanguage-link interwiki-ar"><a href="https://ar.wikipedia.org/wiki/%D8%AA%D8%B1%D8%A7%D8%B5%D9%81_%D8%A7%D9%84%D8%B3%D9%84%D8%B3%D9%84%D8%A9_%D8%A7%D9%84%D9%85%D8%AA%D8%B9%D8%AF%D8%AF" title="تراصف السلسلة المتعدد – Arabic" lang="ar" hreflang="ar" class="interlanguage-link-target">العربية</a></li><li class="interlanguage-link interwiki-es"><a href="https://es.wikipedia.org/wiki/Alineamiento_m%C3%BAltiple_de_secuencias" title="Alineamiento múltiple de secuencias – Spanish" lang="es" hreflang="es" class="interlanguage-link-target">Español</a></li><li class="interlanguage-link interwiki-fa"><a href="https://fa.wikipedia.org/wiki/%D9%87%D9%85%E2%80%8C%D8%AA%D8%B1%D8%A7%D8%B2%D8%B3%D8%A7%D8%B2%DB%8C_%DA%86%D9%86%D8%AF_%D8%AA%D9%88%D8%A7%D9%84%DB%8C" title="هم‌ترازسازی چند توالی – Persian" lang="fa" hreflang="fa" class="interlanguage-link-target">فارسی</a></li><li class="interlanguage-link interwiki-gl"><a href="https://gl.wikipedia.org/wiki/Ali%C3%B1amento_m%C3%BAltiple_de_secuencias" title="Aliñamento múltiple de secuencias – Galician" lang="gl" hreflang="gl" class="interlanguage-link-target">Galego</a></li><li class="interlanguage-link interwiki-ko"><a href="https://ko.wikipedia.org/wiki/%EB%8B%A4%EC%A4%91%EC%84%9C%EC%97%B4%EC%A0%95%EB%A0%AC" title="다중서열정렬 – Korean" lang="ko" hreflang="ko" class="interlanguage-link-target">한국어</a></li><li class="interlanguage-link interwiki-id"><a href="https://id.wikipedia.org/wiki/Persejajaran_sekuen_jamak" title="Persejajaran sekuen jamak – Indonesian" lang="id" hreflang="id" class="interlanguage-link-target">Bahasa Indonesia</a></li><li class="interlanguage-link interwiki-he"><a href="https://he.wikipedia.org/wiki/Multiple_sequence_alignment" title="Multiple sequence alignment – Hebrew" lang="he" hreflang="he" class="interlanguage-link-target">עברית</a></li><li class="interlanguage-link interwiki-ja"><a href="https://ja.wikipedia.org/wiki/%E5%A4%9A%E9%87%8D%E6%95%B4%E5%88%97" title="多重整列 – Japanese" lang="ja" hreflang="ja" class="interlanguage-link-target">日本語</a></li><li class="interlanguage-link interwiki-pt"><a href="https://pt.wikipedia.org/wiki/Alinhamento_m%C3%BAltiplo_de_sequ%C3%AAncias" title="Alinhamento múltiplo de sequências – Portuguese" lang="pt" hreflang="pt" class="interlanguage-link-target">Português</a></li><li class="interlanguage-link interwiki-ru badge-Q17437798 badge-goodarticle" title="good article"><a href="https://ru.wikipedia.org/wiki/%D0%9C%D0%BD%D0%BE%D0%B6%D0%B5%D1%81%D1%82%D0%B2%D0%B5%D0%BD%D0%BD%D0%BE%D0%B5_%D0%B2%D1%8B%D1%80%D0%B0%D0%B2%D0%BD%D0%B8%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5_%D0%BF%D0%BE%D1%81%D0%BB%D0%B5%D0%B4%D0%BE%D0%B2%D0%B0%D1%82%D0%B5%D0%BB%D1%8C%D0%BD%D0%BE%D1%81%D1%82%D0%B5%D0%B9" title="Множественное выравнивание последовательностей – Russian" lang="ru" hreflang="ru" class="interlanguage-link-target">Русский</a></li><li class="interlanguage-link interwiki-sr"><a href="https://sr.wikipedia.org/wiki/Poravnavanje_vi%C5%A1estrukih_sekvenci" title="Poravnavanje višestrukih sekvenci – Serbian" lang="sr" hreflang="sr" class="interlanguage-link-target">Српски / srpski</a></li><li class="interlanguage-link interwiki-sh"><a href="https://sh.wikipedia.org/wiki/Poravnavanje_vi%C5%A1estrukih_sekvenci" title="Poravnavanje višestrukih sekvenci – Serbo-Croatian" lang="sh" hreflang="sh" class="interlanguage-link-target">Srpskohrvatski / српскохрватски</a></li><li class="interlanguage-link interwiki-tr"><a href="https://tr.wikipedia.org/wiki/%C3%87oklu_dizi_hizalamas%C4%B1" title="Çoklu dizi hizalaması – Turkish" lang="tr" hreflang="tr" class="interlanguage-link-target">Türkçe</a></li><li class="interlanguage-link interwiki-zh"><a href="https://zh.wikipedia.org/wiki/%E5%A4%9A%E9%87%8D%E5%BA%8F%E5%88%97%E6%AF%94%E5%B0%8D" title="多重序列比對 – Chinese" lang="zh" hreflang="zh" class="interlanguage-link-target">中文</a></li>					</ul>
				<div class="after-portlet after-portlet-lang"><span class="wb-langlinks-edit wb-langlinks-link"><a href="https://www.wikidata.org/wiki/Special:EntityPage/Q1377767#sitelinks-wikipedia" title="Edit interlanguage links" class="wbc-editpage">Edit links</a></span></div>			</div>
		</div>
				</div>
		</div>
		<div id="footer" role="contentinfo">
							<ul id="footer-info">
											<li id="footer-info-lastmod"> This page was last edited on 6 April 2017, at 09:15.</li>
											<li id="footer-info-copyright">Text is available under the <a rel="license" href="https://en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License">Creative Commons Attribution-ShareAlike License</a><a rel="license" href="https://creativecommons.org/licenses/by-sa/3.0/" style="display:none;"></a>;
additional terms may apply.  By using this site, you agree to the <a href="https://wikimediafoundation.org/wiki/Terms_of_Use">Terms of Use</a> and <a href="https://wikimediafoundation.org/wiki/Privacy_policy">Privacy Policy</a>. Wikipedia® is a registered trademark of the <a href="https://www.wikimediafoundation.org/">Wikimedia Foundation, Inc.</a>, a non-profit organization.</li>
									</ul>
							<ul id="footer-places">
											<li id="footer-places-privacy"><a href="https://wikimediafoundation.org/wiki/Privacy_policy" class="extiw" title="wmf:Privacy policy">Privacy policy</a></li>
											<li id="footer-places-about"><a href="https://en.wikipedia.org/wiki/Wikipedia:About" title="Wikipedia:About">About Wikipedia</a></li>
											<li id="footer-places-disclaimer"><a href="https://en.wikipedia.org/wiki/Wikipedia:General_disclaimer" title="Wikipedia:General disclaimer">Disclaimers</a></li>
											<li id="footer-places-contact"><a href="https://en.wikipedia.org/wiki/Wikipedia:Contact_us">Contact Wikipedia</a></li>
											<li id="footer-places-developers"><a href="https://www.mediawiki.org/wiki/Special:MyLanguage/How_to_contribute">Developers</a></li>
											<li id="footer-places-cookiestatement"><a href="https://wikimediafoundation.org/wiki/Cookie_statement">Cookie statement</a></li>
											<li id="footer-places-mobileview"><a href="https://en.m.wikipedia.org/w/index.php?title=Multiple_sequence_alignment&amp;mobileaction=toggle_view_mobile" class="noprint stopMobileRedirectToggle">Mobile view</a></li>
									</ul>
										<ul id="footer-icons" class="noprint">
											<li id="footer-copyrightico">
							<a href="https://wikimediafoundation.org/"><img src="./Multiple sequence alignment - Wikipedia_files/wikimedia-button.png" srcset="/static/images/wikimedia-button-1.5x.png 1.5x, /static/images/wikimedia-button-2x.png 2x" width="88" height="31" alt="Wikimedia Foundation"></a>						</li>
											<li id="footer-poweredbyico">
							<a href="https://www.mediawiki.org/"><img src="./Multiple sequence alignment - Wikipedia_files/poweredby_mediawiki_88x31.png" alt="Powered by MediaWiki" srcset="/static/images/poweredby_mediawiki_132x47.png 1.5x, /static/images/poweredby_mediawiki_176x62.png 2x" width="88" height="31"></a>						</li>
									</ul>
						<div style="clear:both"></div>
		</div>
		<script>(window.RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgPageParseReport":{"limitreport":{"cputime":"0.520","walltime":"0.570","ppvisitednodes":{"value":2289,"limit":1000000},"ppgeneratednodes":{"value":0,"limit":1500000},"postexpandincludesize":{"value":107105,"limit":2097152},"templateargumentsize":{"value":186,"limit":2097152},"expansiondepth":{"value":6,"limit":40},"expensivefunctioncount":{"value":0,"limit":500},"entityaccesscount":{"value":0,"limit":400},"timingprofile":["100.00%  460.013      1 -total"," 84.14%  387.036      1 Template:Reflist"," 47.82%  219.962     37 Template:Cite_journal"," 14.74%   67.815      5 Template:Cite_web","  9.02%   41.493      1 Template:Good_article","  8.71%   40.087      1 Template:Top_icon","  8.31%   38.213      6 Template:Citation","  6.43%   29.576      1 Template:Category_handler","  3.04%   13.964      2 Template:Cite_book","  0.46%    2.110      1 Template:Column-width"]},"scribunto":{"limitreport-timeusage":{"value":"0.297","limit":"10.000"},"limitreport-memusage":{"value":3180280,"limit":52428800}},"cachereport":{"origin":"mw1184","timestamp":"20170714221232","ttl":2592000,"transientcontent":false}}});});</script><script>(window.RLQ=window.RLQ||[]).push(function(){mw.config.set({"wgBackendResponseTime":83,"wgHostname":"mw1267"});});</script>
	

<div class="suggestions" style="display: none; font-size: 13px;"><div class="suggestions-results"></div><div class="suggestions-special"></div></div><a accesskey="v" href="https://en.wikipedia.org/wiki/Multiple_sequence_alignment?action=edit" style="display: none;"></a></body></html>